# Alternative Splicing Tools
A list of alternative splicing analysis resources

[Review 1](https://pubmed.ncbi.nlm.nih.gov/36999049/)
[Review 2](https://pubmed.ncbi.nlm.nih.gov/36987778/)
[Review 3](https://pubmed.ncbi.nlm.nih.gov/29931155/)
[Review 4](https://academic.oup.com/bib/article/22/6/bbab259/6330938?login=true)
[Review 5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4728800/)

## Splice-aware mapping tools
### 2passtools: a tool for filtering long RNA-Seq alignment from minimap2
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33648554/)
* [GitHub](https://github.com/bartongroup/2passtools)

#### BBMap
* [URL](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbmap-guide/)
* [GitHub](https://github.com/BioInfoTools/BBMap)
* [Paper](http://bib.irb.hr/datoteka/773708.Josip_Maric_diplomski.pdf)

#### BFAST: a BLAT-like mapping tool. Only for SOLiD data.
* [URL](https://sourceforge.net/projects/bfast/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19907642/)
* [GitHub](https://github.com/nh13/bfast)

#### ContextMap2
* [URL](https://www.bio.ifi.lmu.de/software/contextmap/)
* [PubMed](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0557-5)

#### CRAC (Not available)
* [URL](http://crac.gforge.inria.fr/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23537109/)

#### DART
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28968831/)
* [GitHub](https://github.com/hsinnan75/Dart)

#### GSNAP
* [URL](http://research-pub.gene.com/gmap/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20147302/)

#### HISAT2
* [URL](http://daehwankimlab.github.io/hisat2/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31375807/)
* [GitHub](https://github.com/DaehwanKimLab/hisat2)
* [Pubmed HISAT-3N] (https://pubmed.ncbi.nlm.nih.gov/34103331/)

#### iMapSplice: a personalized RNA-Seq alignment tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30096157/)
* [GitHub](https://github.com/LiuBioinfo/iMapSplice)

#### Magic-BLAST
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31345161/)
* [URL](https://ncbi.github.io/magicblast/)
* [GitHub](https://github.com/ncbi/magicblast)

#### MapSplice3
* [URL](http://www.netlab.uky.edu/p/bioinfo/MapSplice2)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20802226/)
* [GitHub](https://github.com/LiuBioinfo/MapSplice)

#### minimap2
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29750242/)
* [GitHub](https://github.com/lh3/minimap2)

#### PALMapper: a splice-aware mapping tool
* [URL](https://bmi.inf.ethz.ch/supplements/palmapper)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21154708/)

#### Rail-RNA: a cloud-enabled splice-aware mapping tool
* [URL](http://rail.bio/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27592709/)
* [GitHub](https://github.com/nellore/rail/releases)

#### RUM: a splice-aware mapping tool
* [URL](http://cbil.upenn.edu/RUM/)
* [GitHub](https://github.com/itmat/rum)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21775302/)

#### segemehl
* [URL](https://www.bioinf.uni-leipzig.de/Software/segemehl/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24626854/)

#### SeqSaw: a splice-aware mapping tool
* [URL - not available](http://bioinfo.au.tsinghua.edu.cn/software/seqsaw/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21575597/)

### Shark: mapping-free tool for fishing relevant reads in an RNA-Seq sample
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32926128/)
* [GitHub](https://github.com/AlgoLab/shark)

#### SpliceMap: a splice-aware mapping tool
* [URL](https://web.stanford.edu/group/wonglab/SpliceMap/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20371516/)

#### Subread: a splice-aware mapping tool
featureCounts is a part of Subread to count features expression
* [URL](http://subread.sourceforge.net/)
* [Bioconductor](http://www.bioconductor.org/packages/release/bioc/html/Rsubread.html)
* [PubMed RSubread](https://pubmed.ncbi.nlm.nih.gov/30783653/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23558742/)

#### STAR: a splice-aware mapping tool
* [GitHub](https://github.com/alexdobin/STAR/releases)
* [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/)

## Alternative splicing and differential splicing detection tools
* [Review 1](https://pubmed.ncbi.nlm.nih.gov/33425250/)
* [Review 2](https://pubmed.ncbi.nlm.nih.gov/35217565/)

### Event-level

### ARH-Seq: a differential splicing detection tool
The R code is in the supplementary materials
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24920826/)

#### ASD: an alternative splicing event detection tool
* [URL](https://soft.novelbio.com/asd/index.html)
* [GitHub](https://github.com/NovelBioCloud/NBCsoftware)
* [PubMed](10.1093/nar/gkt1387)

#### ASDT: an alternative splicing event detection tool
* [URL](http://aias.biol.uoa.gr/~mtheo/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30069446/)

#### ASGAL: an alternative splicing event detection tool
* [URL](https://asgal.algolab.eu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30458725/)
* [GitHub](https://github.com/AlgoLab/galig)

#### ASpediaFI: identification of DAS events and co-regulated genes and pathways
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35085775/)
* [Bioconductor: No documentation](https://bioconductor.org/packages/ASpediaFI)

#### AS-Quant
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33922891/)
* [GitHub](https://github.com/CompbioLabUCF/AS-Quant)

#### ASpli: an alternative splicing event and differential splicing detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33677494/)
* [GitHub](https://github.com/estepi/ASpli)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/ASpli.html)

#### ASprofile: an alternative splicing event detection tool
* [URL](http://ccb.jhu.edu/software/ASprofile/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24555089/)

#### ASTool: AS detection from plant RNA-Seq
* [GitHub](https://github.com/zzd-lab/ASTool)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35456896/)

#### AStrap: an alternative splicing event prediction from transcript sequence
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30535139/)
* [GitHub](https://github.com/BMILAB/AStrap)

#### CASH: a differential splicing analysis tool
* [URL 1](https://soft.novelbio.com/cash/)
* [URL 2](https://sourceforge.net/projects/cash-program/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28387786/)

#### DARTS: an alternative splicing event detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30923373/)
* [GitHub](https://github.com/Xinglab/DARTS)

#### DSGseq: a differential splicing detection tool
* [URL Not available](http://bioinfo.au.tsinghua.edu.cn/software/DSGseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23228854/)

### EBChangePoint
* [URL](http://ebchangepoint.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26873932/)

#### EventPointer: an alternative splicing event detection tool
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/27315794/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/30253752/)
* [PubMed 3](https://pubmed.ncbi.nlm.nih.gov/36128425/)
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/EventPointer.html)

#### FakIR: an intron retention identification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33073477/)
* [GitHub](https://github.com/lbroseus/FakIR)

#### FaRLine: a pipeline for alternative splicing event detection
* [URL](http://kissplice.prabi.fr/pipeline_ks_farline/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29523794/)

#### GESS: an exon skipping detection tool
Based on not supporting tool MISO
* [URL](http://jinlab.net/GESS_Web/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24369421/)

#### iDiffIR: a differential intron retention tool
* [URL](https://combi.cs.colostate.edu/idiffir/introduction.html)
* [Bitbucket](https://bitbucket.org/comp_bio/idiffir/src/master/)

#### IntEREst: an intron retention identification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29642843/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/IntEREst.html)

#### iREAD: an intron retention identification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32028886/)
* [GitHub](https://github.com/genemine/iread)

#### IRFinder: an intron retention identification tool
IRFinder is not maintained
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28298237/)
* [Pubmed IRFinder-S](https://pubmed.ncbi.nlm.nih.gov/34749764/)
* [GitHub](https://github.com/williamritchie/IRFinder)
* [GitHub IRFInder-S](https://github.com/RitchieLabIGH/IRFinder)

#### jSplice: a differential splicing detection tool
Not maintained.
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27153587/)
* [GitHub](https://github.com/yannchristinat/jsplice)

#### JUM: an alternative splicing event detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30104386/)
* [GitHub](https://github.com/qqwang-berkeley/JUM)

#### Keep Me Around: an intron retention identification tool
* [arRxiv](https://arxiv.org/abs/1510.00696)
* [GitHub](https://github.com/pachterlab/kma)

#### KISSPLICE: an alternative splicing event detection tool
KISSDE for differential splicing analysis
* [URL 1](http://kissplice.prabi.fr/)
* [URL 2](http://kissplice.prabi.fr/pipeline_ks_farline/)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/22537044/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/29523794/)
* [Bioconductor KissDE](https://bioconductor.org/packages/release/bioc/html/kissDE.html)

#### MAJIQ: an alternative splicing event detection tool
* [URL](https://majiq.biociphers.org/)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/26829591/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/29236961/)

#### MntJULiP
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36104797/)
* [GitHub](https://github.com/splicebox/MntJulip/)

#### nagnag: a nagnag splicing prediction tool
* [URL](https://sourceforge.net/projects/nagnag/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26028313/)

#### PAVfinder: de novo isoforms inference from transcriptome assemblies
Part of TAP pipeline
* [URL](https://www.bcgsc.ca/resources/software/pavfinder)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30200994/)
* [GitHub](https://github.com/bcgsc/pavfinder)

#### psichomics: an alternative splicing quantification and visualization tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30277515/)
* [Bioconductor](http://bioconductor.org/packages/release/bioc/html/psichomics.html)

#### PSI-Sigma: a differential splicing detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31135034/)
* [GitHub](https://github.com/wososa/PSI-Sigma)

#### PTESFinder: a post-transcriptional exon shuffling
* [URL](https://sourceforge.net/projects/ptesfinder-v1/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26758031/)

#### rMATS: a differential splicing detection
* [URL](http://rnaseq-mats.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25480548/)
* [GitHub](https://github.com/Xinglab/rmats-turbo)

#### SAJR: a differential splicing tool
* [URL Not available](http://storage.bioinf.fbb.msu.ru/~mazin/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23340839/)

### SEASTAR: an alternative transcription start sites detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29546410/)
* [GitHub](https://github.com/Xinglab/SEASTAR)

#### SGSeq: an alternative splicing event detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27218464/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SGSeq.html)

#### splAdder: an alternative splicing event detection tool
* [URL](https://bmi.inf.ethz.ch/supplements/spladder)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/26873928/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/35751815/)
* [GitHub](https://github.com/ratschlab/spladder)

#### SpliceDetector: an alternative splicing event detection tool
* [URL Google Drive](https://drive.google.com/open?id=1dlXKzbvxOH3A85_DVR__V2eI5s16-llv)
* [URL Dropbox](https://www.dropbox.com/s/j5o0og159ig6tej/SpliceDetector%20Executable%20File.rar?dl=0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29567976/)

#### spliceR: a tool for alternative splicing events, differential splicing detection, and isoform switching
* [Bioconductor](http://www.bioconductor.org/packages/2.13/bioc/html/spliceR.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24655717/)

#### SpliceSeq: a tool for investigation of alternative mRNA splicing patterns in data from RNA-seq.
* [URL](https://bioinformatics.mdanderson.org/public-software/spliceseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820202/)

#### SplicingCompass: a differential splicing tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23449093/)
* [GitHub](https://github.com/KoenigLabNM/SplicingCompass)

#### SplicingTypesAnno: an alternative splicing event detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25720307/)
* [URL](https://sourceforge.net/projects/splicingtypes/files/)

#### SUVA
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34152934/)
* [Github](https://github.com/ablifedev/SUVA)

#### TECtool: a terminal codon characterization
* [URL](http://tectool.unibas.ch/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30202060/)
* [GitHub](https://github.com/zavolanlab/TECtool)

#### Whippet: an alternative splicing event detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30220560/)
* [GitHub](https://github.com/timbitz/Whippet.jl)

#### Yanagi: a differential alternative splicing tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31409274/)
* [GitHub](https://github.com/HCBravoLab/yanagi)

### Isoform-level
#### ACTOR: a latent Dirichlet model to compare expressed isoform proportions to a reference panel
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33948626/)
* [GitHub](https://github.com/mccabes292/actor)

#### AERON: an isoform quantification and gene-fusion detection from long reads
* [bioRxiv](https://www.biorxiv.org/content/10.1101/2020.01.27.921338v1)
* [GitHub](https://github.com/SchulzLab/Aeron)

#### Alexa-Seq: an isoform qunification and differential isoform expression tool
Not maintained.
* [URL](http://www.alexaplatform.org/alexa_seq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20835245/)

#### Ballgown
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27560171/)
* [GitHub](https://github.com/alyssafrazee/ballgown)

#### BANDITS: a differential isoform usage
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32178699/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/BANDITS.html)

#### BIISQ: an isoform inference tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29703885/)
* [GitHub](https://github.com/bee-hive/BIISQ)

#### BitSeq: an isoform quantification and differential isoform expression tool
* [PubMed BitSeq](https://pubmed.ncbi.nlm.nih.gov/22563066/)
* [PubMed cjBitSeq](https://pubmed.ncbi.nlm.nih.gov/29091583/)
* [GitHub cjBitSeq](https://github.com/mqbssppe/cjBitSeq)
* [URL](https://code.google.com/archive/p/bitseq/)

#### Cancer DEIso: a differential gene and isoform expression in cancer
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34589189/)
* [URL Not available](http://cosbi4.ee.ncku.edu.tw/DEIso/)

#### casper: an alternative splicing event detection tool
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/casper.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24795787/)

#### CEM: a transcriptome assmbly and isoform quantification tool
* [URL](http://alumni.cs.ucr.edu/~liw/cem.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23060617/)

#### CIDANE: an isoform quantification and reconstruction tool
* [URL](http://ccb.jhu.edu/software/cidane/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30539557/)
* [Bitbucket](https://bitbucket.org/canzar/cidane/src/master/)

#### Cufflinks: a suite of tools for an isoform quantification and differential isoform expression tool
* Cufflinks assembles transcriptomes from RNA-Seq data and quantifies their expression.
* Cuffcompare compares the assembled transcriptomes
* Cuffmerge merges the tranacriptomes from multiple RNA-Seq libraries
* Cuffquant computes the gene and transcript expression profiles 
* Cuffdiff compares expression levels of genes and transcripts in RNA-Seq
* Cuffnorm normalizes the expression levels
* [URL](http://cole-trapnell-lab.github.io/cufflinks/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23222703/)
* [GitHub](https://github.com/cole-trapnell-lab/cufflinks)

#### DiffSplice: a differential isoform expression detection tool
* [URL Not available](http://www.netlab.uky.edu/p/bioinfo/DiffSplice)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23155066/)

#### DRIMseq: a differential isoform expression detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28105305/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/DRIMSeq.html)
* [GitHub Bayes DRIMSeq](https://github.com/mqbssppe/BayesDRIMSeq)

#### DTUrtle: a differential transcript usage for bulk and single-cell RNA-Seq
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34469510/)
* [Github 1](https://github.com/TobiTekath/DTUrtle)
* [GitHub 2](https://tobitekath.github.io/DTUrtle/)

#### EBseq: a differential isoform expression detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23428641/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/EBSeq.html)

#### Event Analysis: an isoform inference (a conda recipe)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30021829/)
* [GitHub](https://github.com/McIntyre-Lab/events)

#### FASE
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36682028/)
* [GitHub](https://github.com/harshsharma-cb/FASE)

#### SEVA: a differential isoform expression
SEVA is implemented in the R package GSReg
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29342249/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/GSReg.html)

#### HBA-DEALS: a differential gene and isoform expression
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32660516/)
* [GitHub](https://github.com/TheJacksonLaboratory/HBA-DEALS)

#### iReckon: an isoform inference and quantification tool
* [URL](http://compbio.cs.toronto.edu/ireckon/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23204306/)

#### IsoEm2: an isoform quantification tool
* [URL](https://dna.engr.uconn.edu/?page_id=105)
* [GitHub](https://github.com/mandricigor/isoem2)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21504602/)

#### IsoformSwitchAnalyzer: an isoform switch detection
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30989184/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/IsoformSwitchAnalyzeR.html)

#### IsoformUsage
The list of R scripts
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35042461/)
* [GitHub](https://github.com/ZXiaopu/IsoformUsage/tree/main/dataAnalysis)

#### IsoInfer: a tool for isoforms inference
* [URL Not available](http://www.cs.ucr.edu/%E2%88%BCjianxing/IsoInfer.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21385036/)

#### iso-kTSP: an isoform switch detection
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25578962/)
* [BitBucket](https://bitbucket.org/regulatorygenomicsupf/iso-ktsp/src/master/)

#### IsoSeq de novo: an isoform identification from IsoSeq 
The list of Python scripts
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28316149/)
* [GitHub](https://github.com/liuxiaoxian/IsoSeq_AS_de_novo)

#### IsoSplitter: an isofrom quantification for long reads
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34021065/)
* [GitHub](https://github.com/Hengfu-Yin/IsoSplitter)

#### ISP: a tool for isoforms inference
* [URL](http://alumni.cs.ucr.edu/~liw/isp.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25708199/)

#### IUTA: a differential isoform expression tool
* [URL](https://www.niehs.nih.gov/research/resources/software/biostatistics/iuta/index.cfm)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25283306/)

#### kallisto: an isoform quantification tool
sleuth is a differential isoform expression tool after kallisto analysis

* [URL kallisto](https://pachterlab.github.io/kallisto/about)
* [URL sleuth](http://pachterlab.github.io/sleuth/about)
* [PubMed kallisto](https://pubmed.ncbi.nlm.nih.gov/27043002/)
* [PubMed sleuth](https://pubmed.ncbi.nlm.nih.gov/28581496/)
* [GitHub kallisto](https://github.com/pachterlab/kallisto)

### LORALS
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/35922509/)
* [URL](https://github.com/LappalainenLab/lorals)

#### metaDIEA: a differential isoform expression analysis
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34698224/)
* [GitHub](https://github.com/BiodataAnalysisGroup/metaDIEA)

#### MMSEQ: an isoform quantification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21310039/)
* [GitHub](https://github.com/eturro/mmseq)

#### MISO: a differential isoform expression tool
Not maintained
* [URL](http://hollywood.mit.edu/burgelab/miso/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21057496/)
* [GitHub](https://github.com/yarden/MISO)

#### MIETIE: an isoform inference and quantification tool
* [URL](https://bmi.inf.ethz.ch/supplements/mitie)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23980025/)
* [GitHub](https://github.com/ratschlab/MiTie)

#### MINTIE
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34686194/)
* [GitHub](https://github.com/Oshlack/MINTIE)

#### mtim: an isoform inference tool
* [arXiv](https://arxiv.org/abs/1309.5211)
* [GitHub](https://github.com/nicococo/mTIM)

### NBBt-test: a versatile method for differential analysis of multiple types of RNA-seq data
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/35896555/)
* [URL](https://CRAN.R-project.org/package=NBBttest)

#### NBSplice: a differential isoform usage
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31972288/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/NBSplice.html)

#### NLDMSeq: an isoform quantification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26475308/)
* [GitHub](https://github.com/PUGEA/NLDMseq)

#### NURD: an isoform quantification tool
* [URL Not available](http://bioinfo.au.tsinghua.edu.cn/software/NURD/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23837734/)

#### PennDiff: a differential isoform usage detection
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29474557/)
* [GitHub](https://github.com/tigerhu15/PennDiff)

#### PRAM: transcript models inference from pooled RNA-seq
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32958497/)
* [Bioconductor](http://bioconductor.org/packages/release/bioc/html/pram.html)
* [GitHub](https://github.com/pliu55/pram)

#### PSGInfer: an isoforms inference and differential analysis tool
* [URL](http://deweylab.biostat.wisc.edu/psginfer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23846746/)

#### QuickIsoSeq
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33835441/)
* [GitHub](https://github.com/shanrongzhao/QuickIsoSeq)

#### rDiff: a differential isoform expression detection tool
* [URL](https://public.bmi.inf.ethz.ch/user/drewe/rdiff/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23585274/)
* [GitHub](https://github.com/ratschlab/rDiff)

#### RSEM: an isoform quantification tool
The variant of the tools is pRSEM: an isoform quantification with a complementary data
* [URL RSEM](https://deweylab.github.io/RSEM/)
* [URL pRSEM](https://deweylab.github.io/pRSEM/)
* [PubMed RSEM](https://pubmed.ncbi.nlm.nih.gov/21816040/)
* [PubMed pRSEM](https://pubmed.ncbi.nlm.nih.gov/27405803/)
* [GitHub](https://github.com/deweylab/RSEM)

#### rSeqDiff: a differential isoform expression detection tool
* [URL](https://jhui2014.github.io/rseqdiff/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24260225/)

#### rSeqNP: a differential isoform expression detection tool
* [URL](https://jhui2014.github.io/rseqnp/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25717189)

#### Sailfish: an isoform quantification tool
* [URL](https://sailfish.readthedocs.io/en/master/sailfish.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24752080/)
* [GitHub](https://github.com/kingsfordgroup/sailfish)

#### salmon: an isoform quantification tool
* [URL](https://combine-lab.github.io/salmon/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28263959/)
* [GitHub](https://github.com/COMBINE-lab/salmon)

#### SDEAP: a differential isoform expression detection tool for population data
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27522083/)
* [GitHub](https://github.com/ewyang089/SDEAP/wiki)

#### SLIDE: an isoform inference and quantification tool
No documentation
* [URL](http://www.stat.ucla.edu/~jingyi.li/software-and-data.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22135461/)

#### Sparselso: an isoform identification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28968634/)
* [GitHub](https://github.com/henryxushi/SparseIso)

#### SpliceGrapher: a tool for creating splice graphs and isoforms predicting
* [URL](http://splicegrapher.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22293517/)

#### spliceR: a tool for alternative splicing, differential detection and isoform switching
* [Bioconductor](http://www.bioconductor.org/packages/2.13/bioc/html/spliceR.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24655717/)

#### SplicingFactory: an isoform diversity
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34499147/)
* [Bioconductor]https://www.bioconductor.org/packages/release/bioc/html/SplicingFactory.html

#### StringTie: a transcriptome assembly and isoform quantification tool
* [URL](http://ccb.jhu.edu/software/stringtie/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25690850/)

#### subgraphquant: a transcript abundance estimator
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/35041494/)
* [Pubmed 2](https://pubmed.ncbi.nlm.nih.gov/33971903/)
* [GitHub](https://github.com/Kingsford-Group/subgraphquant)

#### SUPPA: an isoform quantification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26179515/)
* [PubMed SUPPA2](https://pubmed.ncbi.nlm.nih.gov/29571299/)
* [GitHub](https://github.com/comprna/SUPPA)
* [BitBucket](https://bitbucket.org/regulatorygenomicsupf/suppa/src/master/)

#### SwitchSeq
Archived
* [GitHub](https://github.com/mgonzalezporta/SwitchSeq)
* [bioRxiv](https://www.biorxiv.org/content/10.1101/005967v1)

#### tappAS: a functional effect of alternative splicing
* [URL](https://app.tappas.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32423416/)

#### TIGAR2: an isoform quantification tool from longer RNA-Seq
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/23821651/)
* [PibMed 2](https://pubmed.ncbi.nlm.nih.gov/25560536/)
* [GitHub](https://github.com/nariai/tigar2)

#### Traph: an isoform quantification tool
* [URL](https://www.cs.helsinki.fi/en/gsa/traph/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734627/)

#### TRIMD: Iso-Seq, RNA-Seq, and deepCAGE data integration for transcriptome resolution 
The list of Perl scripts
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27407110/)
* [GitHub](https://github.com/flemingtonlab/TRIMD)

### Exon-level
#### DEXSeq: a differential exon usage tool
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/DEXSeq.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722343/)

#### DIEGO: a differential alternative splicing tool
* [URL](http://www.bioinf.uni-leipzig.de/Software/DIEGO/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29088309/)

#### DJExpress: An Integrated Application for Differential Splicing Analysis and Visualization
* [GitHub](https://github.com/MauerLab/DJExpress)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36304260/)

#### diffUTR: a differential exon usage analysis
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33849458/)
* [Bioconductor](http://bioconductor.org/packages/release/bioc/html/diffUTR.html)

#### ExClust: a package for constitutive exons analysis
Part of the sysSeq package. No easily accessible documentation
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23360225/)
* [GitHub](https://github.com/ellispatrick/sydSeq)

#### ExCluster: a differential exon expression tool
* [BioConductor](https://bioconductor.org/packages/release/bioc/html/ExCluster.html)

#### FineSplice: a TopHat2 wrapper to identify expressed junctions
* [URL](https://sourceforge.net/projects/finesplice/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24574529/)

#### FreePSI: an exon-inclusion ratio quantification tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29136203/)
* [GitHub](https://github.com/JY-Zhou/FreePSI)

#### HMMSplicer: a splice junctions detection
* [URL Not available](http://derisilab.ucsf.edu/index.php?software=105)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21079731/)

#### HTSeq: an RNA-Seq quantification tool
The tool can quantify exon expression

* [URL](https://htseq.readthedocs.io/en/master/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25260700/)
* [GitHub](https://github.com/htseq/htseq)

#### iGEMS: a model for alternative exon usage detection 
An R script
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27095197/)
* [GitHub](https://github.com/iaingallagher/iGEMS_scripts)

#### Intron-Essentiality
A list of scripts
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33906938/)
* [GitHub](https://github.com/evendrow/Intron-Essentiality/)

#### iTAD: a switch-like exons identification 
The code is upon request
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28542625/)

#### JunctionSeq: a differential exon and junctions usage
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27257077/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/JunctionSeq.html)

#### LeafCutter: an intron quantification and differential analysis
* [URL LeafViz](https://leafcutter.shinyapps.io/leafviz/)
* [URL](http://davidaknowles.github.io/leafcutter/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29229983/)
* [GitHub](https://github.com/davidaknowles/leafcutter)

### McSplicer
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33515239/)
* [GitHub](https://github.com/canzarlab/McSplicer)
* [Zenodo](https://doi.org/10.5281/zenodo.4449881)

#### MicroExonator
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33482885/)
* [GitHub](https://github.com/hemberg-lab/MicroExonator)

#### MISO: a differential isoform expression tool
The tool also detects differentially regulated exons.
MISO also presents sashimi plots for exon expression visualisation.
No longer maintained.
* [URL](http://hollywood.mit.edu/burgelab/miso/)
* [PubMed MISO](https://pubmed.ncbi.nlm.nih.gov/21057496/)
* [PubMed sashimi plots](https://pubmed.ncbi.nlm.nih.gov/25617416/)
* [GitHub](https://github.com/yarden/MISO)

#### PAIRADISE: a paired differential isoform expression
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32781045/)
* [GitHub](https://github.com/Xinglab/PAIRADISE)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/PAIRADISE.html)

#### Portcullis: a splice junctions detection
* [URL Documemntation](https://portcullis.readthedocs.io/en/latest/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6302956/)
* [GitHub](https://github.com/TGAC/portcullis)

#### Read-Split-Fly: a splice junctions detection (base on Read-Split-Walk and Read-Split-Run)
* [PubMed RSW](https://www.ncbi.nlm.nih.gov/pubmed/24991935/)
* [PubMed RSR](https://pubmed.ncbi.nlm.nih.gov/27556805/)
* [GitHub RSR](https://github.com/xuric/read-split-run)
* [GitHub RSF](https://github.com/kinnejeff/read-split-fly)

#### RNAprof: a differential RNA processing tool
* [URL](http://rssf.i2bc.paris-saclay.fr/Software/rnaprof.php)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26849165/)

#### SeqGSEA: a gene set enrichment analysis with differential expression and splicing
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SeqGSEA.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734663/)
* [GitHub](https://github.com/sunlightwang/SeqGSEA_1.13)

#### SigFuge: a differential isoform expression detection tool
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/SigFuge.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25030904/)

#### SpliceJumper: a splice junctions detection
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26678515/)
* [GitHub](https://github.com/simoncchu/SpliceJumper)

#### SpliceLauncher: a splice junctions detection
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31617569/)
* [GitHub](https://github.com/raphaelleman/SpliceLauncher)

#### SPLICE-q: an intron splicing efficiency
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34266387/)
* [GitHub](https://github.com/vrmelo/SPLICE-q)

#### SpliceTrap: a tool to quantify exon inclusion levels using paired-end RNA-seq data
Redirects to ESEfinder
* [URL](http://rulai.cshl.edu/splicetrap/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21896509/)

#### SpliSER: a splice site usage quantification
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34017946/)
* [Github](https://github.com/CraigIDent/SpliSER)

#### SplicingGraphs: a splicing graphs creation, visualization, and quantification tool
* [GitHub](https://github.com/Bioconductor/SplicingGraphs)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SplicingGraphs.html)

## sQTL analysis
[Review 1](https://pubmed.ncbi.nlm.nih.gov/35960994/)

### FIVEx: a eQTL and sQTL browser
* [URL](https://fivex.sph.umich.edu)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34459872/)
* [Github](https://github.com/statgen/fivex)

### GLiMMPS: an sQTL detection
* [GutHub](https://github.com/Xinglab/GLiMMPS)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23876401/)

### PVAAS: a tool to identify SNVs associated with aberrant splicing
* [URL](http://pvaas.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25573917/)

### SNPlice: variants that modulate Intron retention from RNA-sequencing data
* [PubMed]( 10.1093/bioinformatics/btu804)
* [URL Not available](https://code.google.com/p/snplice/)

### sQTLseekeR: an sQTL detection
* [GitHub](https://github.com/guigolab/sQTLseekeR)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25140736/)

### SplicePlot: an sQTL visualization
No easily accessible documentation
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/24363378/)
* [GitHub](https://github.com/wueric/SplicePlot)

### ulfasQTL: an sQTL detection (based on sQTLseekeR)
* [URL Not available](http://bioinfo.au.tsinghua.edu.cn/software/ulfasQTL/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28198669/)

## Splice sites prediction and cryptic splicing
* [Review 1](https://pubmed.ncbi.nlm.nih.gov/36987778/)

### AUGUSTUS
* [URL new](https://bioinf.uni-greifswald.de/webaugustus/index)
* [URL old](https://bioinf.uni-greifswald.de/augustus/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21216780/)

### CADD-splice: a variant effect on alternative splicing
* [URL](https://cadd.gs.washington.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33618777/)

### ChotStitch
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29300846/)
* [GutHub](https://github.com/bcgsc/ChopStitch)
  
### CI-SpliceAI: machine learning predictions of disease causing splicing variants
* [URL](https://ci-spliceai.com)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35657932/)
* [GitHub](https://github.com/YStrauch/CI-SpliceAI__Annotation)

### COSSMO: an alternative splicing site usage prediction
The code for the model
* [URL](http://cossmo.genes.toronto.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29949959/)
* [GitHub](https://github.com/PSI-Lab/COSSMO)

### CRYP-SKIP: an effect of exonic mutation
* [URL](https://cryp-skip.img.cas.cz/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/19142208)

### CrypSplice: a cryptic splice variations prediction tool
* [URL](http://liuzlab.org/software/crypsplice/#hoem)

### DASSI: splice identification from DNA sequences
The model
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33588916/)
* [GitHub](https://github.com/shabirmoosa/DASSI)

### DeepASmRNA
* [URL](http://cmb.bnu.edu.cn/DeepASmRNA/index.php/download)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36325068/)
* [GitHub](https://github.com/CMB-BNU/DeepASmRNA)

### DeepDSSR: a donor splice site recognition 
The model
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31349311/)

### Deep Splicing Code: an alternative splicing event prediction from sequence
The code
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31374967/)
* [URL Not available](https://home.jbnu.ac.kr/NSCL/dsc.htm)
* [GitHub](https://github.com/louadi/DSC)

### DNABERT: pre-trained Bidirectional Encoder Representations from Transformers model for DNA-language in genome
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33538820/)
* [GitHub 1](https://github.com/jerryji1993/DNABERT)
* [GitHub 2](https://github.com/Zhihan1996/DNABERT_2)

### EX-SKIP: a variant effect prediction on alternative splicing
* [URL](https://ex-skip.img.cas.cz/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/21309043)

### Exon ByPASS: predicting exon criticality
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35286381/)
* [GitHub](https://github.com/wavelifescience/ExonByPass)

### GeneSplicer: a splice site prediction
* [URL](https://www.cbcb.umd.edu/software/GeneSplicer/gene_spl.shtml)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/11222768/)

### Genomic-Kmer-Surprisal-Model
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33194378/)
* [GitLab](https://gitlab.com/cruk-mi/genomic-kmer-surprisal-model)

### HAL: an effect of variants on alternative splicing
* [URL](http://splicing.cs.washington.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26496609/)

### Human Splicing Finder: a splice site prediction 
Limited access
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19339519/)
* [URL](https://www.genomnis.com/access-hsf)

### IntSplice2
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34349788/)
* [URL](https://www.med.nagoya-u.ac.jp/neurogenetics/IntSplice2)

### KipoiSplice
* [bioRxiv](https://www.biorxiv.org/content/10.1101/375345v1)
* [URL](http://kipoi.org/models/KipoiSplice/)

### MaxEntScan: a tool for human 5'splice sites scoring
* [URL](http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html)

### MLCsplice
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35976049/)
* [URL Not available](http://39.105.51.3:8090/MLCsplice/)

### MiSplice
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29617666/)
* [GitHub](https://github.com/ding-lab/misplice)

### MMSplice: a tool for prediction of the effect of genetic variants on alternative splicing
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30823901/)
* [GitHub](https://github.com/gagneurlab/MMSplice_MTSplice)
* [Kipoi](http://kipoi.org/models/MMSplice/)

### MTSplice
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33789710/)
* [GitHub](https://github.com/gagneurlab/MMSplice_MTSplice)

### MutationTaster
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33893808/)
* [URL](https://www.genecascade.org/MutationTaster2021/)

### NetAspGene: a splice site prediction in Aspergillus
* [URL](https://services.healthtech.dtu.dk/service.php?NetAspGene-1.0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/18948220/)

### NetGene2 Server: a splice site prediction
* [URL 1](http://www.cbs.dtu.dk/services/NetGene2/)
* [URL 2](https://services.healthtech.dtu.dk/service.php?NetGene2-2.42)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/2067018/)

### NetUTR: a splice site prediction in 5'UTR regions of human genes
* [URL](https://services.healthtech.dtu.dk/service.php?NetUTR-1.0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/14960723/)

### NNSPLICE: a splice site prediction by neural network
* [URL](https://www.fruitfly.org/seq_tools/splice.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/9278062/)

### PEPSI: a prediction of PSI value changes due to genetic variants
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31074545/)
* [GitHub](https://github.com/rwang916/PEPSI)

### PredPSI-SVR: a prediction of PSI value changes due to genetic variants
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31070294/)
* [GitHub](https://github.com/chenkenbio/PredPSI-SVR)

### RegTools: Integrated analysis of genomic and transcriptomic data for the discovery of splice-associated variants in cancer
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36949070/)
* [GitHub](https://github.com/griffithlab/regtools)

### rPGA: a hidden splicing variations
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28766294/)
* [GitHub](https://github.com/Xinglab/rPGA)

### SAVNet: a tool for detection of effect of genetic variants on alternative splicing
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31687339/)
* [GitHub](https://github.com/friend1ws/SAVNet)

### SiLVA: prediction of synonymous (silent) mutations within the human genome
* [URL](http://compbio.cs.toronto.edu/silva/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23736532/)

### SKIPPY: a tool for detection of exonic variants that modulate splicing
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20158892)
* [URL](https://research.nhgri.nih.gov/skippy/download.shtml)

### SPIDEX
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/25525159/)
* [URL](http://www.openbioinformatics.org/annovar/spidex_download_form.php)

### Splice2Deep: a splice site prediction
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32550561/)
* [GitHub](https://github.com/SomayahAlbaradei/Splice_Deep)

### SpliceAI: a tool for detection of the effect of genetic variants on alternative splicing
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30661751/)
* [GitHub](https://github.com/Illumina/SpliceAI)

### Spliceator: a splice site prediction
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34814826/)
* [GitLab](https://git.unistra.fr/nscalzitti/spliceator)
* [URL](http://www.lbgi.fr/spliceator/)

### Spliceman: a web tool for prediction of sequence variations in splicing
* [URL Not available](http://fairbrother.biomed.brown.edu/spliceman/index.cgi)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22328782/)

### SpliceView: a splice site prediction
* [URL Not available](http://bioinfo.itb.cnr.it/~webgene/wwwspliceview.html)

### SpliceViNCI: a non-canonical splice sites prediction
No code provided
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34088258/)

### SPANR: a splicing-based analysis of SNVs
* [URL Not available](http://tools.genes.toronto.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25525159/)

### SQUIRLS: a tool that detects changes in splicing regulatory sequences
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34289339/)
* [GitHub](https://github.com/TheJacksonLaboratory/Squirls)

### TraP: transcript inferred pathogenecity score
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/28794409/)
* [URL](http://trap-score.org/)

### Veridical
* [PubMed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3983938/)
* [URL Not available](http://www.veridical.org/defaultsite)
 
## Alternative splicing regulation
### AIRBP: Identification of RNA-binding proteins using machine-learning techniques
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33685590/)
* [URL Not available](http://cs.uno.edu/∼tamjid/Software/AIRBP/code_data.zip)

### BEAM
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27580722/)
* [URL](http://beam.uniroma2.it/home)

### BPP: a branch point prediction
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28633445/)
* [GitHub](https://github.com/zhqingit/BPP)

### branchpointer: a branch point prediction
* [bioRxiv](https://www.biorxiv.org/content/10.1101/094003v1)
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/branchpointer.html)

### CapR
* [URL Not available]()
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24447569/)

### DanQ (for DNA)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27084946/)
* [GitHub](https://github.com/uci-cbcl/DanQ)

### DeepRKE: prediction of RBP binding sites
No documentation
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33334313/)
* [GitHub](https://github.com/youzhiliu/DeepRKE/)

### DeepBind
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26213851/)
* [URL Not available](http://tools.genes.toronto.edu/deepbind/)

### DeeperBind (For DNA binding)
* [GitHub](https://github.com/hassanzadeh/DeeperBind)
* [Paper](https://pure.ulster.ac.uk/en/publications/proceedings-of-2018-ieee-international-conference-on-bioinformati)

### DeepBtoD: RNA-binding proteins prediction via integrated deep learning
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35451938/)
* [URL Not available](http://175.27.228.227/DeepBtoD/)

### deepnet-rbp
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26467480/)
* [GitHub](https://github.com/thucombio/deepnet-rbp)

### dlprb
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30423078/)
* [GitHub](https://github.com/ilanbb/dlprb)

### dsRBPBind: the effect of RNA secondary structure on double-stranded RNA-protein binding
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34668517/)
* [URL Not available]http://bioserv.mps.ohio-state.edu/dsRBPBind

### dSreg: a differential splicing and regulation prediction
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31834368/)
* [Bitbucket](https://bitbucket.org/cmartiga/dsreg/src/master/)

### ESEFinder: a web tool for exon splicing enhancers identification
* [URL](http://krainer01.cshl.edu/cgi-bin/tools/ESE3/esefinder.cgi?process=home)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824367/)

### ExonScan: a regulatory exonic motifs identification 
The code is upon request
* [URL](http://hollywood.mit.edu/burgelab/software.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/15607979/)

### ExonSuite: a tool for PUF binding regions analysis
The Python script with no documentation
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/23816174/)
* [GitHub](https://github.com/dilanustek/ExonSuite)

### GraphProt
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24451197/)
* [URL](http://www.bioinf.uni-freiburg.de/Software/GraphProt/)
* [GitHub](https://github.com/dmaticzka/GraphProt)

### iDeep
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28245811/)
* [GitHub](https://github.com/xypan1232/iDeep)

### iDeepE
* [GitHub](https://github.com/xypan1232/iDeepE)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29722865/)

### iDeepS
* [GitHub](https://github.com/xypan1232/iDeepS)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29970003/)

### iDeepV
* [GitHub](https://github.com/xypan1232/iDeepV)
* [Paper](https://www.sciencedirect.com/science/article/abs/pii/S0925231218304685)

### iONMF
* [GitHub](https://github.com/mstrazar/iONMF)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26787667/)

### LaBranchoR: a branchpoint prediction tool
Last release: 2019, February, 20th
* [URL](http://bejerano.stanford.edu/labranchor/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30224349/)
* [GitHub](https://github.com/jpaggi/labranchor)

### MEMERIS
* [URL Not available](http://www.bioinf.uni-freiburg.de/~hiller/MEMERIS/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16987907/)

### MIRA: a tool for mutation identification for RNA Alterations
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29592900/)
* [GitHub](https://github.com/comprna/MIRA)

### ModCon: a splice site usage modification tool
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34136105/)
* [GitHub](https://github.com/caggtaagtat/ModCon)

### Oli
No code
* [https://pubmed.ncbi.nlm.nih.gov/24780077/]

### RBPmap: a tool for mapping binding sites of RNA binding proteins
Last release: 2021, February
* [URL](http://rbpmap.technion.ac.il/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24829458/)

### RCK
* [URL](https://cb.csail.mit.edu/cb/rck/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27307637/)
  
### RNAcommender
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27503225/)
* [URL not available](http://rnacommender.disi.unitn.it/)

### RNAcontext
* [URL Not available](http://morrislab.med.utoronto.ca/software)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20617199/)

### rMAPS2: a tool for RNA-binding protein analysis
* [URL](http://rmaps.cecsresearch.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32286627/)

### PCB: approach to infer the dynamic regulatory relationships between alternative splicing events 
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36930678/)
* [URL](http://hkumath.hku.hk/~wkc/PCB(data+code).zip)

### RNABP: a branchpoint prediction
* [URL Not available](http://nsclbio.jbnu.ac.kr/tools/RNABP/)

### SVM-BPfinder: a branchpoint prediction tool
Last release: 2016
* [URL Not available](http://regulatorygenomics.upf.edu/Software/SVM_BP/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21124863/)

### WDFSMF: an RNA-AS target prediction
No documentation
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34410342/)
* [Github](https://github.com/yushanqiu/WDFSMF)

### Zeng et al.
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27307608/)
* [URL Not available](http://cnn.csail.mit.edu/)

## Proteomics

[Review 1](https://pubmed.ncbi.nlm.nih.gov/36845399/)

### jcast: takes in alternative splicing events and returns custom protein sequence databases for isoform analysis
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31825849/)
* [PubMed (Protocol)](https://pubmed.ncbi.nlm.nih.gov/33377032/)
* [GitHub](https://github.com/ed-lau/jcast)

### Long-Read-Proteogenomics
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35241129/)
* [GitHub](https://github.com/sheynkman-lab/Long-Read-Proteogenomics)

### QUILTS: the identification of novel proteins, resulting from single nucleotide variants, splice variants, and fusion genes
* [URL Not available](http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26631509/)
* [GitHub](https://github.com/ekawaler/pyQUILTS/) - on a very early stage

### PASS: an alternative splicing detection from proteome
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31095856/)
* [GitHub](https://github.com/wupengomics/PASS)

### pgdb and pypgatk: creation of proteogenomics databases based on ENSEMBL resources
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34904638/)
* [GitHub pypgatk](https://github.com/bigbio/py-pgatk/)
* [URL pgdb](https://nf-co.re/pgdb)

### SpliceVista: a tool for splice variant identification and visualization in shotgun proteomics data
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24692640/)
* [GitHub](https://github.com/yafeng/SpliceVista)

### SPLICIFY: a proteogenomic pipeline for differential splice variant identification
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28747380/)
* [GitHub](https://github.com/NKI-TGO/SPLICIFY)

### TopPG: a proteogenomic tool for generating proteoform sequence databases with genetic alterations and alternative splicing events
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33183009/)
* [URL Not available](http://proteomics.informatics.iupui.edu/software/toppg/)

## Network analysis
### ALT-IN: predicting protein interaction network perturbation by alternative splicing
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34818539/)
* [GitHub](https://github.com/KorkinLab/alt-in-tool)
* [Zenodo](https://zenodo.org/record/5234256)
* [DockerHub](https://hub.docker.com/r/narykov/alt-in)

### CoSpliceNet: a co-splicing network tool
* [URL](http://delasa.github.io/co-spliceNet/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27793091/)
* [GitHub](https://github.com/Delasa/co-spliceNet)

### DeepIII: an isoform-isoform interaction prediction
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33764878/)
* [URL Not available](http://mlda.swu.edu.cn/codes.php?name=DeepIII)

### DIGGER: a role of alternative splicing in protein-protein interactions
* [URL](https://exbio.wzw.tum.de/digger/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32976589/)

### DomainGraph: effect of alternative splicing on domain-domain interactions
* [URL](https://domaingraph.bioinf.mpi-inf.mpg.de/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20513647/)

### Iso-Net
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/32534132/)
* [GitHub](https://github.com/Dingjie-Wang/Iso-Net)

### NEASE
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34857024/)
* [GitHub](https://github.com/louadi/NEASE) 

## Single-cell data (much much more here https://www.scrna-tools.org/)
### acorde: networks of isoform co-usage from single-cell data
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35383181/)
* [GitHub](https://github.com/ConesaLab/acorde)

### BRIE: an isoform quantification tool
* [PubMed BRIE](https://pubmed.ncbi.nlm.nih.gov/28655331/)
* [Pubmed BRIE2](https://pubmed.ncbi.nlm.nih.gov/34452629/)
* [GitHub](https://github.com/huangyh09/brie)
 
### DESJ
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33674641/)
* [GitHub](https://github.com/liushang17/DESJ-detection)

### Expedition: an alternative splicing detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28673540/)
* [GitHub](https://github.com/YeoLab/Expedition)

### IsoCell: an approach for isoform-based single-cell clustering
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35100120/)
* [GitHub](https://github.com/genemine/IsoCell)
* [URL](https://zenodo.org/record/4395707)

### MARVEL: an integrated alternative splicing analysis platform for single-cell RNA sequencing data
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36631981/)
* [URL](https://cloud.r-project.org/web/packages/MARVEL/index.html)

### Psix: splicing that changes across a landscape of single cells
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35858747/)
* [GitHub](https://github.com/lareaulab/Psix)

### satuRn
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36762203/)
* [GitHub](https://github.com/statOmics/satuRn)
  
### scAllele: detection and analysis of variants in scRNA-seq
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36054357/)
* [GitHub](https://github.com/gxiaolab/scAllele/)

### SICILIAN: splice junctions identification in bulk and single RNA-Seq
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34353340/)
* [GitHub](https://github.com/salzmanlab/SICILIAN)

### SingleSplice: an alternative splicing detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26740580/)
* [GitHub](https://github.com/jw156605/SingleSplice)

### SpliZ: a splicing quantification score for single-cell
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35241832/)

## Long reads (much much more here https://long-read-tools.org/)

[Review 1](https://pubmed.ncbi.nlm.nih.gov/37066125/)
[Review 2](10.1186/s13059-020-1935-5)

### BRCA-LRseq-pipeline
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35044822/)
* [GitHub](https://github.com/TheJacksonLaboratory/BRCA-LRseq-pipeline)

### brca-isoforms: long-read isoform analysis platform and sequencing resource for breast cancer
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35044822/)
* [URL](https://brca-isoforms.jax.org/)

### Cupcake

### deSALT: an alignment tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31842925/)
* [GitHub](https://github.com/ydLiu-HIT/deSALT)

### flair
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32188845/)
* [GitHub](https://github.com/BrooksLabUCSC/flair)

### FLAME: a bioinformatics pipeline for alternative splicing analysis of gene-specific or transcriptome-wide long-read sequencing data
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34253685/)
* [GitHub](https://github.com/marabouboy/FLAME)

### FLAMES

### Freddie: annotation-independent detection and discovery of transcriptomic alternative splicing isoforms using long-read sequencing
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36478271/)
* [GitHub](https://github.com/vpc-ccg/freddie/)

### IDP

### IsoAnnot

### IsoCon

### IsoSeq3

### IsoSplitter

* [GitHub](https://github.com/Hengfu-Yin/IsoSplitter)

### IsoTV

### L-RAPiT
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36555493/)
* [GitHub](https://github.com/Theo-Nelson/long-read-sequencing-pipeline)

### LIQA
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36993612/)
* [Github 1](https://github.com/yuntianf/Longcell-pre)
* [GitHub 2](https://github.com/yuntianf/Longcell)

### LORALS

### Longcell
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34140043/)
* [GitHub](https://github.com/WGLab/LIQA)

### Mandalorian

### NanoSplicer

### Pinfish

### scallop-LR: a transcritome assembler
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36099042/)
* [GitHub](https://github.com/ylab-hi/ScanExitronLR)

### ScanExitronLR
* [bioRxiv](https://www.biorxiv.org/content/10.1101/632703v1)
* [GitHub](https://github.com/Kingsford-Group/scallop/tree/isoseq)

### scNaST

### SiT
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36928528/)

### SpliceHunter
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29508290/)
* [Bitbucket](https://bitbucket.org/canzar/splicehunter/src/master/)

### SQANTI

### TAMA

### TALON
* [bioRxiv](https://www.biorxiv.org/content/10.1101/672931v2)
* [GitHub](https://github.com/mortazavilab/TALON)

### TAPIS

### VIsoQLR
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36881176/)
* [GitHub](https://github.com/TBLabFJD/VIsoQLR)

### WUB

### uLTRA
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34302453/)
* [GitHub](https://github.com/ksahlin/ultra)

## Isoform Function Inference/Functional interpretation

### ASpediaFI: Functional Interaction Analysis of Alternative Splicing Events
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35085775/)
* [Bioconductor](https://bioconductor.statistik.tu-dortmund.de/packages/3.13/bioc/html/ASpediaFI.html)

### DeepIDA: isoform-disease associations prediction
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33571094/)
* [URL Not available](http://mlda.swu.edu.cn/codes.php?name=DeepIDA)

### DeepIsoFun
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/30535380/)
* [GitHub](https://github.com/dls03/DeepIsoFun/)

### DIFFUSE
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31510699/)
* [GitHub](https://github.com/haochenucr/DIFFUSE)

### DisoFun
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32176770/)
* [URL Not available](https://mlda.swu.edu.cn/codes.php?name=DisoFun)
  
### DMIL-IsoFun
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34282449/)
* [URL](http://www.sdu-idea.cn/codes.php?name=DMIL-Isofun)

### factR: Functional Annotation of Custom Transcriptomes
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35895263/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/factR.html) 

### iMILP
* [PubMed](10.1093/nar/gkt1362)
* [URL Not available](http://zhoulab.usc.edu/IsoFP)

### IsoDA: an isoform-disease association prediction
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33826865/)
* [URL Not available](http://mlda.swu.edu.cn/codes.php?name=IsoDA)

### IsoFun
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31250882/)
* [URL Not available](http://mlda.swu.edu.cn/codes.php? name=IsoFun)

### IsoPred
* [PubMed](10.1371/journal.pcbi.1003314)
* [URL Not available](http://guanlab.ccmb.med.umich.edu/isoPred/.)

### isopretEM
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36929917/)
* [GitHub](https://github.com/TheJacksonLaboratory/isopretEM)

### IsoResolve
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/32966552/)
* [GitHub](https://github.com/genemine/IsoResolve)

### NMD Classifier: a nonsense mediated decay classifier
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28369084/)
* [URL](https://sourceforge.net/projects/transcriptome-analysis/files/)

### n1pas: an alternatevely spliced pathway detection
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31143202/)
* [GitHub](https://github.com/grizant/n1pas/tree/master)

### PathwaySplice: a pathway analysis for alternative splicing genes
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29688305/)
* [GitHub](https://github.com/SCCC-BBC/PathwaySplice)
* [Bioconductor](http://bioconductor.riken.jp/packages/3.8/bioc/html/PathwaySplice.html)

### SAPFIR: A webserver for the identification of alternative protein features
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35751026/)
* [URL](https://bioinfo-scottgroup.med.usherbrooke.ca/sapfir/)
* [GitHub](https://github.com/DelongZHOU/SAPFIR)

### SpliceTools: a suite of downstream RNA splicing analysis tools to investigate mechanisms and impact of alternative splicing
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36864749/)
* [GitHub](https://github.com/flemingtonlab/SpliceTools)

### TRIFID
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34046593/)
* [Bitbucket](https://gitlab.com/bu_cnio/trifid)

### Trinotate
* [GitHub](https://github.com/Trinotate/Trinotate)

### TS-Isofun: a tissue-specific isoform function prediction
No code available
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34185647/)

### WLRM
The code is not available
* [Paper](https://dl.acm.org/doi/pdf/10.1145/3097983.3097984)

## Circular RNA
* [Review 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7986655/)
* [Review 2 Circular RNA databases](https://pubmed.ncbi.nlm.nih.gov/34195960/)
* [Review 1 Circular RNA databases](https://pubmed.ncbi.nlm.nih.gov/31998941/)

### At-C-RNA: Arabidopsis thaliana CircRNAs
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34788390/)
* [URL](https://plantcircrna.ibch.poznan.pl/) 

### AQUARIUM
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34115093/)
* [Github](https://github.com/wanjun-group-seu/AQUARIUM)

### CircAST
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32007626/)
* [GitHub](https://github.com/xiaofengsong/CircAST)

### Circall
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34645386/)
* [GitHub](https://github.com/datngu/Circall)
* [URL](https://www.meb.ki.se/sites/biostatwiki/circall)

### CIRCExplorer
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33588028/)
* [GitHub](https://github.com/YangLab/CIRCexplorer)
* [CIRCexplorer2](https://circexplorer2.readthedocs.io/en/latest/)

### CircIMPACT
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34356060/)
* [Github](https://github.com/AFBuratin/circIMPACT)

### CirComPara2
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34698333/)
* [GitHub](https://github.com/egaffo/CirComPara2)

### Circ-LocNet: localisation prediction
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35897818/)
* [URL](https://circ_rna_location_predictor.opendfki.de/)

### CircSplice
* [URL](http://gb.whu.edu.cn/CircSplice/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30849979/)
* [GitHub](https://github.com/GeneFeng/CircSplice)

### circRNA-sponging: a pipeline for extensive analysis of circRNA expression and their role in miRNA sponging
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36789427/)
* [GitHub](https://github.com/biomedbigdata/circRNA-sponging)

### Ciri
* [URL](https://sourceforge.net/projects/ciri/)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/27350239/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/30660194/)

### DEBKS: A Tool to Detect Differentially Expressed Circular RNA
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33631429/)
* [GitHub](https://github.com/yangence/DEBKS)

### DMC-CDA
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34103016/)
* [Github](https://github.com/zzl1996zzl/DMC-CDA)

### find_circ
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/23446348/)
* [GitHub](https://github.com/marvin-jens/find_circ)

### FUCHS
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28265491/)
* [GitHub](https://github.com/dieterich-lab/FUCHS)

### GreenCircRNA
* [URL Not available](http://greencirc.cn/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/32510565/)

### JEDI
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34252942/)
* [Github](https://github.com/hallogameboy/JEDI)

### PCirc
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33407069/)
* [GitHub](https://github.com/Lilab-SNNU/Pcirc)

### PlantCircNet:
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/31725858/)
* [URL](https://biokeanos.com/source/PlantCircNet)

### psirc
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34663689/)

### SRCP
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34556162/)
* [GitHub](https://github.com/avigayel/SRCP)

### TransCirc: an interactive database for translatable circular RNAs based on multi-omics evidence
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33074314/)
* [URL](https://www.biosino.org/transcirc/)

### VirusCircBase: a database of virus circular RNAs
* [URL](http://www.computationalbiology.cn/ViruscircBase/home.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/32349124/)

## Aberrant Splicing
### FRASER
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33483494/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/37066374/)
* [Bioconductor](http://bioconductor.org/packages/release/bioc/html/FRASER.html)

#### LeafCutter: an intron quantification and differential analysis
* [URL LeafViz](https://leafcutter.shinyapps.io/leafviz/)
* [URL](http://davidaknowles.github.io/leafcutter/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29229983/)
* [GitHub](https://github.com/davidaknowles/leafcutter)

## Pipelines
### 3D RNA-Seq: a pipeline
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33345702/)
* [URL](https://3drnaseq.hutton.ac.uk/app_direct/3DRNAseq/)
* [GitHub](https://github.com/wyguo/ThreeDRNAseq)

### AIDD: a pipeline for transcript level analysis
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33375933/)
* [GitHub](https://github.com/RNAdetective/AIDD)

### AltAnalyze: an alternative splicing event detection tool
Kallisto-Splice is part of a tool for a splice-aware mapping
* [URL](http://www.altanalyze.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20513647/)
* [GitHub](https://github.com/nsalomonis/altanalyze)
 
### Bisbee
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34031440/)
* [GitHub](https://github.com/tgen/bisbee)

### DICAST
* [GitHub](https://github.com/CGAT-Group/DICAST)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/37260511/)

### OneStopRNAseq: a pipeline for RNA-Seq analysis
* [URL](https://mccb.umassmed.edu/OneStopRNAseq/index.php)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33023248/)

### PPLine: a pipeline for SNP calling and gene/isoform quantification
* [URL](https://sourceforge.net/projects/ppline/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26147802/)

### PipeOne-NM: RNA-Seq Analysis Pipeline for Non-Model Organisms
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/37239350/)
* [GitHub](https://github.com/Lisijun-m/pipeone-nm)

### RAP: a web pipeline for RNA-Seq analysis
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33835454/)
* [URL Not available](https://bioinformatics.cineca.it/rap/index.php)

### rmappet: a pipeline for alternative splicing analysis
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36854218/)
* [URL](https://github.com/didrikolofsson/rmappet/)

### SeqCVIBE: Interactive Analysis, Exploration, and Visualization of RNA-Seq
For RNA-Seq only
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35314664/)
* [GitHub](https://github.com/hybridstat/seqcvibe )

## Other

### ACEScan: the algorithm for prediction of conserved alternatively spliced exons from pairs of conserved mouse/human exons 
The code is upon request
* [URL](http://hollywood.mit.edu/burgelab/software.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/15708978/)

### ASES: a tool for assessing the impact of alternative splicing, initiation and termination of transcription on protein diversity in evolution
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35188186/)
* [URL]( http://www.lcqb.upmc.fr/Ases)

### ASimulatoR: RNA-Seq reads with splicing simulation
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33647976/)
* [GitHub](https://github.com/biomedbigdata/ASimulatoR)

### ASNEO: a detection of alternative splicing neoantigens
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32697765/)
* [GitHub](https://github.com/bm2-lab/ASNEO)

### BEERS2: RNA-Seq reads with splicing simulation
* [PubMed](10.1101/2023.04.21.537847)
* [GitHub](https://github.com/itmat/BEERS2)

### CAMPAREE: an RNA expression simulator
* [Pubmed] (https://pubmed.ncbi.nlm.nih.gov/34563123/)
* [GitHub] (https://github.com/itmat/CAMPAREE)

### CHESSBOARD: Splicing-based subtyping
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36599821/)
* [URL](https://chessboard.readthedocs.io/en/latest/index.html)

### EGIO: orthologous exons detection
Last release
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/35946527/)
* [GitHub](https://github.com/wu-lab-egio/EGIO)

### Enhanced Integrated Gradients: a method to identify specific features
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32560708/)
* [Bitbucket](https://bitbucket.org/biociphers/eig/src/master/)

### ERPIN: a mitochondrial intron detection
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35369492/)

### eSkip-Finder: a machine learning-based web application and database to identify the optimal sequences of antisense oligonucleotides for exon skipping
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34104972/)
* [URL](https://eskip-finder.org)

### ESPRNN: epigenome-based splicing prediction
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32584815/)
* [GitHub](https://github.com/gersteinlab/esprnn)

### Ex-Ex Primer: designing oligonucleotides spanning spliced nucleic acid regions
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34756973/)
* [URL](http://resource2.ibab.ac.in/exprimer/)

### ExoPLOT
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35863675/)
* [URL](http://retrogenomics3.uni-muenster.de:3838/exz-plot-d/)

### ExOrthist: an exon evolution
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34416914/)
* [Github](https://github.com/biocorecrg/ExOrthist)

### ExTraMapper: Exon- and Transcript-level mappings for orthologous gene pairs
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34014317/)
* [GitHub](https://github.com/ay-lab/ExTraMapper)
* [URL](http://ay-lab-tools.lji.org/extramapper)

### FSOM: a methylation pattern associated with alternative splicing
* [URL](https://sourceforge.net/projects/fsom/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26365234/)

### IsoDeconvMM: cell type deconvolution based on isoform composition
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34921386/)
* [GitHub](https://github.com/hheiling/IsoDeconvMM)

### IsoSel: an isoform phylogenetic analysis tool
* [URL](http://doua.prabi.fr/software/isosel)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28323858/)

### Matt: feature extraction for alternative splicing
* [URL](http://matt.crg.eu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30010778/)

### MBS: a genome browser annotation track for microRNA binding sites in the whole human transcriptome
* [URL](https://datasharingada.fondazionerimed.com:8080/MBS)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/37114805/)

### Mirage: multiple sequence isoform alignment
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31080963/)
* [GitHub](https://github.com/TravisWheelerLab/Mirage)

### MOCCASIN: a correction for confounders
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34099673/)
* [Bitbucket](https://bitbucket.org/biociphers/moccasin/src/master/)

### PIC-Me: a classification model between paralogs and isoforms
* [Pubmed](10.1186/s12859-021-04229-x)

### Reboot:
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34316711/)
* [GitHub](https://github.com/galantelab/reboot)

### rnabridge-denovo: de novo Bridging Paired-end RNA-seq Data
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/37033458/)
* [GitHub](https://github.com/Shao-Group/rnabridge-denovo)

### ORQAS: an alternative splicing quantification tool from RNA-Seq and Ribo-Seq
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32286305/)
* [GitHub](https://github.com/comprna/orqas)

### PrimerSeq: a tool for RT-PCR primers design that evaluates alternative splicing events
* [URL](http://primerseq.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24747190/)
* [GitHub](https://github.com/ctokheim/PrimerSeq)

### SCISSOR: a framework for identifying structural changes in RNA transcripts
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33436599/)
* [GitHub](https://github.com/hyochoi/SCISSOR)

### SimSpliceEvol: an alternative splicing-aware simulation of biological sequence evolution
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31842741/)
* [GitHub](https://github.com/UdeS-CoBIUS/SimSpliceEvol)

### SLIDR and SLOPPR: flexible identification of spliced leader trans-splicing and prediction of eukaryotic operons from RNA-Seq data
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33752599/)
* [GitHub](https://github.com/wenzelm/slidr-sloppr)

### SplicePie: an order of splicing prediction
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25800735/)
* [GitHub](https://github.com/pulyakhina/splicing_analysis_pipeline)

### ThorAxe: an evolutionary splicing graphs construction
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34266979/)

### TranscriptAchilles: a transcript-level biomarkers for cancer
* [URL](https://biotecnun.unav.es/app/TranscriptAchilles)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30942869/)
* [GitLab](https://gitlab.com/fcarazo.m/transcriptachilles)

### UNCOVER: the algorithm for predicting conserved alternatively spliced exons (and retained introns) in orthologous intron pairs (Upon request)
* [URL](http://hollywood.mit.edu/burgelab/software.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16110330/)

## Databases
### AceView: a database of alternative splicing isoforms
* [URL](https://www.ncbi.nlm.nih.gov/IEB/Research/Acembly/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16925834/)

### AltTrans: a database of alternative splicing and alternative polyadenilation (archived)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16556303/)
* [FTP](ftp://ftp.ebi.ac.uk/pub/databases/astd)

### APPRIS: a database of principal splice isoforms
* [URL](http://appris.bioinfo.cnio.es/#/)
* [PubMed 1](https://www.ncbi.nlm.nih.gov/pubmed/23161672)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/34755885/)

### Arabidopsis PeptideAtlas
* [URL](http://www.peptideatlas.org/builds/arabidopsis/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34411258/)

### ASCancer Atlas: a comprehensive knowledgebase of alternative splicing in human cancers
* [URL](https://ngdc.cncb.ac.cn/ascancer)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36318242/)

### AS-CMC: a pan-cancer database of alternative splicing for molecular classification of cancer
* [URL](https://www.pmrc.re.kr/ASCMC/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36473963/)

### ASCOT
* [URL](http://ascot.cs.jhu.edu/)
* [GitHub](https://github.com/jpling/ascot)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31919425/)

### ASlive/### LivestockExp
* [URL](https://bioinfo.njau.edu.cn/livestockExp)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/36253723/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/32000661/)

### ASpedia: a database of human alternative splicing
* [URL](http://combio.snu.ac.kr/aspedia/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29106599/)

### ASPicDB: a database of alternative splicing patterns in human genes
* [URL](http://srv00.recas.ba.infn.it/ASPicDB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21051348/)

### ASPN: a splicing-derived neoepitopes database
* [URL](http://jianglab.org.cn/ASPN/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34285345/)

### Blood Proteoform Atlas
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35084980/)
* [URL](https://blood-proteoform-atlas.org/)

### CancerSplicingQTL: a database of SNPs that affect alternative splicing
* [URL](http://www.cancersplicingqtl-hust.com/#/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30329095/)

### CanIsoNet: a database to study the functional impact of isoform switching events in diseases
* [URL](https://www.caniso.net)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/37123454/)

### CASA: a comprehensive database resource for the COVID-19 Alternative Splicing Atlas
* [URL](http://www.splicedb.net/casa/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36266726/)

### CAS-viewer: a visualization tool of cancer alternatively splicing
* [URL](http://genomics.chpc.utah.edu/cas/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29697367/)

### CattleGTEx
* [URL Not available](http://cgtex.roslin.ed.ac.uk/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35953587/)

### CuAS: a database of alternative splicing in cucumber
* [URL](http://cmb.bnu.edu.cn/alt_iso/index.php)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32183712/)

### dbNSFP v4: a comprehensive database of transcript-specific functional predictions and annotations for human nonsynonymous and splice-site SNVs
* [URL](http://database.liulab.science/dbNSFP)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33261662/)
 
### DBTSS: a database of transcription start sites
* [URL](https://dbtss.hgc.jp/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/29126224)

### eQTL Catalogue: a database of eQTL and sQTL
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34493866/)
* [URL](https://www.ebi.ac.uk/eqtl/)

### ExonSkipAD: exon skipping in Alzheimer's disease
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33497435/)
* [URL](https://ccsm.uth.edu/ExonSkipAD/)

### ExonSkipDB: a functional characterization of skipped exons
* [URL](https://ccsm.uth.edu/ExonSkipDB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31642488/)

### FishExp: A comprehensive database and analysis platform for gene expression and alternative splicing of fish species
* [URL](https://bioinfo.njau.edu.cn/fishExp)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35891795/)

### FungiExp
* [URL](https://bioinfo.njau.edu.cn/fungiExp)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36655761/)

### Gene Expression Nebulas (GEN): a comprehensive data portal integrating transcriptomic profiles across multiple species at both bulk and single-cell levels
* [URL](https://ngdc.cncb.ac.cn/gen/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34591957/)

### H-DBAS: a human alternative splicing database
* [URL](http://h-invitational.jp/h-dbas/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/19969536?dopt=Abstract)

### HEXEvent: a database of Human EXon splicing Events
* [URL](http://hexevent.mmg.uci.edu/cgi-bin/HEXEvent/HEXEventWEB.cgi)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23118488/)
 
### The Human Proteoform Atlas
* [URL](http://human-proteoform-atlas.org)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/34986596/)

### IntroVerse: a comprehensive database of introns across human tissues
* [URL](https://rytenlab.com/browser/app/introverse)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36399497/)

### iRBase
* [URL](https://irbase.igh.cnrs.fr/)
* [PubMed](10.1186/s13059-017-1184-4)

### ISOdb: a database of Iso-Seq isoforms
* [URL Not available](http://isodb.xieslab.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30581839/)

### ISOexpresso: a database of isoform expression in cancer
* [URL](http://wiki.tgilab.org/ISOexpresso/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27519173/)

### Kassiopeia: a database of mutually exclusive exonomes
* [URL](https://kassiopeia.motorprotein.de/kassiopeia)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24507667/)

### LncAS2Cancer: alternative splicing in lncRNAs in cancer
* [URL Not available](https://lncrna2as.cd120.com/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32820322/)

### MeDAS: a Metazoan Developmental Alternative Splicing database
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33084905/)
* [URL](https://das.chenlulab.com/#/)
  
### MetazExp
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34718719/)
* [URL](https://bioinfo.njau.edu.cn/metazExp)

### MutSpliceDB: A database of splice sites variants with RNA-seq based evidence on effects on splicing
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33600011/)
* [URL](https://brb.nci.nih.gov/splicing)

### OncoSplicing
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34554251/)
* [URL](http://www.oncosplicing.com/)

### oRNAment: a database of putative RNA binding protein target sites in the transcriptomes of model species
* [URL](http://rnabiology.ircm.qc.ca/oRNAment/)
* [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7145663/)

### Pan-cancer repository of validated natural and cryptic mRNA splicing mutations
* [URL](https://validsplicemut.cytognomix.com/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31275557/)

### PID: Plant Intron Database
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34111777/)
* [URL Not available](http://biodb.sdau.edu.cn/PID/index.php)

### PlaASDB: a comprehensive database of plant alternative splicing events in response to stress
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/37106367/)
* [URL](http://zzdlab.com/PlaASDB/ASDB/index.html)

### Plant Alternative Splicing Database
* [URL](http://proteomics.ysu.edu/altsplice/)

### PlantExp
* [URL Not available](https://biotec.njau.edu.cn/plantExp)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36271793/)

### PlantSPEAD: a splicing-related protein expression database
* [URL](http://chemyang.ccnu.edu.cn/ccb/database/PlantSPEAD/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33010087/)

### PISE: Plant Intron-Splicing Efficiency Database
* [URL](https://plantintron.com/)
* [PubMed](10.1007/s11427-022-2193-3)
  
### PolyA_DB: a database of pre-mRNA cleavage and polyA sites
* [URL](https://exon.apps.wistar.org/PolyA_DB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29069441/)

### ProSAS: a database of the effects of alternative splicing events on the structure of the resulting protein isoforms.
* [URL Not available](https://drupal.bio.ifi.lmu.de/forschung/structural-bioinformatics/prosas/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/17933774)

### RJunBase: a database of RNA splice junctions in human normal and cancerous tissues
* [URL](http://www.RJunBase.org)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33179749/)

### RBPDB
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21036867/)
* [URL](http://rbpdb.ccbr.utoronto.ca/)

### recount3: an online resource consisting of RNA-seq gene, exon, and exon-exon junction counts as well as coverage
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34844637/)
* [URL](http://rna.recount.bio/)

### SASD: the Synthetic Alternative Splicing Database for identifying novel isoform from proteomics
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/24267658/)
* [URL] Does not work

### S-CAP: a list of splice-related pathogenicity scores
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/30804562/)
* [URL](http://bejerano.stanford.edu/scap/)
* [Bitbucket](https://bitbucket.org/bejerano/splicing_classifier/)

### SilkBase: an integrated transcriptomic and genomic database for Bombyx mori and related species
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/35670730/)
* [URL](https://silkbase.ab.a.u-tokyo.ac.jp)

### SpliceAid: a database of experimentally proven RNA target motifs bound by splicing proteins in human
* [URL](http://www.introni.it/splicing.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19261717/)
* [URL](http://193.206.120.249/splicing_tissue.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21922594/)

### SpliceVault
* [URL](https://kidsneuro.shinyapps.io/splicevault/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/36747048/)

### SpliVap
* [URL Not working properly](http://splicing.bioinformatica.crs4.it/)

### TassDB: a tandem splice sites database
* [URL 1](http://www.tassdb.info/)
* [URL 2](http://tassdb2.leibniz-fli.de/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20429909)

### TCGA SpliceSeq: splicing patterns alternation in TCGA
* [URL](https://bioinformatics.mdanderson.org/TCGASpliceSeq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26602693/)

### TeaAS: a database of alternative splicing in tea plants
* [URL](http://www.teaas.cn/index.php)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34154536/)

### Transposable element-derived alternative splicing in cancer
* [URL](http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&hubUrl=http://jordan.biology.gatech.edu/teAs/hub.txt)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32075558/)

### TSVdb: TCGA splicing variants database
* [URL](http://www.tsvdb.com/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29843604/)
* [GitHub](https://github.com/wenjie1991/TSVdb)

### VastDB: a database of alternative splicing profiles
* [URL](https://vastdb.crg.eu/wiki/Main_Page)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28855263/)
* [PubMed VAST-TOOLS](https://pubmed.ncbi.nlm.nih.gov/35895261/)

## Visualization
### DoChaP: a domain-splicing visualization
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/33988713/)
* [URL](https://dochap.bgu.ac.il/)

### GFFView: a RNA-Seq viewer
* [URL Not available](http://120.27.116.110/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28355123/)

### Integrated Genome Browser: a visual analysis of alternative splicing
* [URL](https://www.bioviz.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24792048/)

### Jutils: A visualization toolkit for differential alternative splicing events
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34021745/)
* [GitHub](https://github.com/Splicebox/Jutils)

### ggsashimi: an alternative splicing visualization
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30118475/)
* [GitHub](https://github.com/guigolab/ggsashimi)

### Manananggal: a visualization of alternative splicing events
* [PubMed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5319012/)
* [GitHub](https://github.com/barannm/Manananggal)

### RNASeqViewer: a visualization tool for gene expression and isoforms
* [URL Not available](http://bioinfo.au.tsinghua.edu.cn/software/RNAseqViewer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24215023/)

### ScisorWiz: visualizing differential isoform expression in single-cell long-read data
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/35604081/)
* [GitHub](http://github.com/ans4013/ScisorWiz)

#### SpliceV: a splicing visualization
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31068132/)
* [GitHub](https://github.com/flemingtonlab/SpliceV)

### TraC: a visualization a shared between splice variants sequences
* [URL](http://labs.pathology.jhu.edu/nauen/trac/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29763731/)
* [GitHub](https://github.com/nauenlab/TraC)

### VALERIE: visualizing splicing at single-cell
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32898151/)
* [CRAN](https://cran.r-project.org/web/packages/VALERIE/index.html)

#### Vials: a visualization of alternative splicing tool
* [URL Not available](http://vials.io/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26529712/)
* [GitHub (archived)](https://github.com/Caleydo/vials)

## Structures
### surfaltr: An R/Bioconductor package to benchmark surface protein isoforms by rapid prediction and visualization of transmembrane topologies

* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35678367/)
* [Bioconductor](https://bioconductor.org/packages/surfaltr)
 
## Transcript assembly
#### BayesDenovo: a transcriptome assembly
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34480063/)
* [GitHub](https://github.com/henryxushi/BayesDenovo)

#### CLASS2: a transcriptome assembly tool
* [URL](https://sourceforge.net/projects/splicebox/files/)
* [PubMed CLASS](https://pubmed.ncbi.nlm.nih.gov/23734605/)
* [PubMed CLASS2](https://pubmed.ncbi.nlm.nih.gov/26975657/)

#### ConSemble: a tool for trascriptome assembly improvement
* [Pubmed] (https://pubmed.ncbi.nlm.nih.gov/34674629/)
* [URL] (http://bioinfolab.unl.edu/emlab/consemble/ )
 
#### isoLASSO: a transcriptome assembly and isoform quantification tool
* [URL](http://alumni.cs.ucr.edu/~liw/isolasso.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21951053/)

#### Jumper: a transcriptome assembly tool
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34795232/)
* [GitHub](https://github.com/elkebir-group/Jumper)

#### MultiTrans: a transcriptome assembly
* [Pubmed] (https://pubmed.ncbi.nlm.nih.gov/34033544/)
* [GitHub] (https://github.com/jzbio/MultiTrans)

#### Oases: a transcriptome assembly tool
* [URL Not available](https://www.ebi.ac.uk/~zerbino/oases/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22368243/)

#### rnaSPAdes: a transcriptome assembly tool
* [URL](http://cab.spbu.ru/software/rnaspades/)
* [bioRxiv](https://www.biorxiv.org/content/10.1101/2020.01.24.918482v1)
* [GitHub](https://github.com/ablab/spades)

#### Scripture: a transcriptome assembly tool
* [URL Not available](http://software.broadinstitute.org/software/scripture/home)

#### SOAPdenovo-trans: a transcriptome assembly tool
* [URL](https://sourceforge.net/projects/soapdenovotrans/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24532719/)
* [GitHub](https://github.com/aquaskyline/SOAPdenovo-Trans)

#### Tiglon: a trascriptome assembly
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/35355524/)
* [GitHub](https://github.com/yutingsdu/Tiglon-v.1.1)

#### TRANS-Abyss: a transcriptome assembly tool
* [URL](https://www.bcgsc.ca/resources/software/trans-abyss)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20935650/)
* [GitHub](https://github.com/bcgsc/transabyss)

#### TransRef: a transcriptome assembly
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/34254977/)
* [Sourceforge](https://sourceforge.net/projects/transassembly/files/)

#### Trinity: a transcriptome assembly tool
* [URL](https://sourceforge.net/projects/trinityrnaseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21572440/)

#### Velvet: a transcriptome assembly tool
* [URL Not available](https://www.ebi.ac.uk/~zerbino/velvet/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/18349386/)
* [GitHub](https://github.com/dzerbino/velvet/tree/master)

## Time-series
#### Spycone
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/36579860/)
* [GitHub](https://github.com/yollct/spycone) 

#### TSIS: a time-series isoform switches detection tool
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29028262/)
* [GitHub](https://github.com/wyguo/TSIS)


## Benchmark studies

### Splice variants
* [Comparison of in silico strategies to prioritize rare genomic variants impacting RNA splicing for the diagnosis of genomic disorders](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8523691/)
* [Benchmarking deep learning splice prediction tools using functional splice assays](https://pubmed.ncbi.nlm.nih.gov/33942434/)
* [Performance evaluation of computational methods for splice-disrupting variants and improving the performance using the machine learning-based framework](https://pubmed.ncbi.nlm.nih.gov/35976049/)

### Splicing
* [Computational comparison of common event-based differential splicing tools: practical considerations for laboratory researchers](https://pubmed.ncbi.nlm.nih.gov/34174808/)
* [A comprehensive benchmarking of differential splicing tools for RNA-seq analysis at the event level](https://pubmed.ncbi.nlm.nih.gov/37020334/)
* [Alternative splicing analysis benchmark with DICAST](https://pubmed.ncbi.nlm.nih.gov/37260511/)
* [Retained introns in long RNA-seq reads are not reliably detected in sample-matched short reads](https://pubmed.ncbi.nlm.nih.gov/36369064/)
* [Exploring differential exon usage via short- and long-read RNA sequencing strategies](https://pubmed.ncbi.nlm.nih.gov/36168804/)
