# Alternative-Splicing-List
A list of alternative splicing analysis resources
## Splice-aware mapping tools
#### RUM: a splice-aware mapping tool
Last release: 2015, January, 27th
* [URL](http://cbil.upenn.edu/RUM/)
* [GitHub](https://github.com/itmat/rum)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21775302/)

#### Scipio: a BLAT-based alignment tool to determine the gene structure given a protein sequence and a genome sequence
Last release: 2013, May, 24th
* [URL](https://www.webscipio.org/webscipio/download_scipio)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/18554390/)

#### UnSplicer: a splice-aware alignment and junction detection tool
Last release: 2013, November, 25th
* [URL](http://exon.gatech.edu/Genemark/paul/unsplicer/index.htm)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24259430/)

## Alternative splicing and differential splicing detection tools
### Event-level

#### DSGSeq: a differential splicing detection tool
Last release: 2012, May, 14th
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/DSGseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23228854/)

#### KISSPLICE: an alternative splicing event detection tool
Last release: 2020, July, 2nd
* [URL](http://kissplice.prabi.fr/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22537044/)

#### rMATS: a differential splicing detection
Last release: 2021, February, 8th
* [URL](http://rnaseq-mats.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25480548/)
* [GitHub](https://github.com/Xinglab/rmats-turbo)

#### SeqGSEA: a gene set enrichment analysis with differential expression and splicing
Last release: 2018, December, 12th (the last commit)

* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SeqGSEA.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734663/)
* [GitHub](https://github.com/sunlightwang/SeqGSEA_1.13)

#### SpliceSeq: a tool for investigation of alternative mRNA splicing patterns in data from RNA-seq.
The tool detects events, compares between samples and predicts the consequence on a protein structure

Last release: 2014, February, 21st
* [URL](https://bioinformatics.mdanderson.org/public-software/spliceseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820202/)

#### SplicingCompass: a diffrential splicing tool
Last release: 2019, October, 24th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23449093/)
* [GitHub](https://github.com/KoenigLabNM/SplicingCompass)

#### Webscipio: a web tool for prediction of eukaryotic gene structures
* [URL](https://www.webscipio.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21718515/)

### Isoform-level
#### DiffSplice: a differential isoform expression detection tool
Last release: 2013, January, 1st
* [URL](http://www.netlab.uky.edu/p/bioinfo/DiffSplice)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23155066/)

#### Integrated Genome Browser: a visual analysis of alternative splicing
Last release: 2020, December, 3rd 
* [URL](https://www.bioviz.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24792048/)

#### IsoEm2: an isoform quantification tool
Last release: 2017, April, 3rd
* [URL](https://dna.engr.uconn.edu/?page_id=105)
* [GitHub](https://github.com/mandricigor/isoem2)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21504602/)

#### IsoInfer: a tool for isoforms inference
Last release: 2010, September, 9th
* [URL](http://www.cs.ucr.edu/%E2%88%BCjianxing/IsoInfer.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21385036/)

#### MMSEQ: an isoform quantification tool
Last release: 2019, November, 11th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21310039/)
* [GitHub](https://github.com/eturro/mmseq)

#### NURD: an isoform quantification tool
Last release: _not found_. Published in 2013.
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/NURD/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23837734/)

#### PSGInfer: an isoforms inference and differential analysis tool
Last release: 2013, October, 25th
* [URL](http://deweylab.biostat.wisc.edu/psginfer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23846746/)

#### RNASeqViewer: a visualization tool for gene expression and isoforms
Last release: 2013, October, 14th
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/RNAseqViewer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24215023/)

#### rSeqDiff: a differential isoform expression detection tool
Last release: _not found_. Published in 2013
* [URL](https://jhui2014.github.io/rseqdiff/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24260225/)

#### SpliceGrapher: a tool for creating splice graphs and isoforms predicting
Last release: 2014, April, 24th
* [URL](http://splicegrapher.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22293517/)

#### TIGAR2: an isoform quantification tool from longer RNA-Seq
Last release: 2015, February, 19th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23821651/)
* [GitHub](https://github.com/nariai/tigar2)

#### Traph: an isoform quantification tool
Last release: 2014, September, 2nd
* [URL](https://www.cs.helsinki.fi/en/gsa/traph/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734627/)

### Exon-level
#### DEXSeq: a differential exon usage tool
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/DEXSeq.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722343/)

#### ExClust: a package for constitutive exons analysis
Part of the sysSeq package.

Last release: 2015, July, 17th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23360225/)
* [GitHub](https://github.com/ellispatrick/sydSeq)

#### ExCluster: a differential exon expression tool
Last release: _not found_. Published in 2020.
* [BioConductor](https://bioconductor.org/packages/release/bioc/html/ExCluster.html)

#### SpliceTrap: a tool to quantify exon inclusion levels using paired-end RNA-seq data

Last release: 2013, May, 17th
* [URL](http://rulai.cshl.edu/splicetrap/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21896509/)

## sQTL analysis
### GLiMMPS: an sQTL detection
Last release: 2013, March, 12th
* [GutHub](https://github.com/Xinglab/GLiMMPS)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23876401/)

### SplicePlot: an sQTL visualization
Last release: 2013, October, 15th

* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/24363378/)
* [GitHub](https://github.com/wueric/SplicePlot)

## Alternative splicing regulation
### ExonSuite: a tool for PUF binding regions analysis
Last release: 2013, January, 13th (the last commit)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/23816174/)
* [GitHub](https://github.com/dilanustek/ExonSuite)

## Databases
### AceView: a database of alternative splicing isoforms
Last update: 2012, October, 16th

Organisms: human, mouse, rat, worm, arabidopsis
* [URL](https://www.ncbi.nlm.nih.gov/IEB/Research/Acembly/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16925834/)

### AltTrans: a database of alternative splicing and alternative polyadenilation (archived)
Last update: 2008, September, 5th

Organisms: human, mouse, rat
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16556303/)
* [FTP](ftp://ftp.ebi.ac.uk/pub/databases/astd)

### APPRIS: a database of principal splice isoforms
Last update: 2021

Organisims: 19 vertebrate and invertebrate genomes
* [URL](http://appris.bioinfo.cnio.es/#/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23161672)

### ASPicDB: a database of alternative splicing patterns in human genes
Last update: 2017, May, 4th, 196,446 splicing events	

Organism: human
* [URL](http://srv00.recas.ba.infn.it/ASPicDB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21051348/)

### DBTSS: a database of transcription start sites
Last update: 2017, December, 21st

Organism: human
* [URL](https://dbtss.hgc.jp/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/29126224)

### H-DBAS: a human alternative splicing database
Last update: 2011, December, 26th

Organism: human
* [URL](http://h-invitational.jp/h-dbas/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/19969536?dopt=Abstract)

### HEXEvent: a database of Human EXon splicing Events
Last update: 2017, September, 29th

Organism: human
* [URL](http://hexevent.mmg.uci.edu/cgi-bin/HEXEvent/HEXEventWEB.cgi)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23118488/)

### PolyA_DB: a database of pre-mRNA cleavage and polyA sites
Last update: 2018, August. 107 human samples, 246 mouse samples, 11 rat samples, 9 chicken samples.

Organisms: human, mouse, rat, chicken
* [URL] (https://exon.apps.wistar.org/PolyA_DB/)
* [PubMed] (https://pubmed.ncbi.nlm.nih.gov/29069441/)

### ProSAS: a database of the effects of alternative splicing events on the structure of the resulting protein isoforms.
Last update: _not found_. Published in 2008.

* [URL](https://drupal.bio.ifi.lmu.de/forschung/structural-bioinformatics/prosas/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/17933774)

### SpliceAid2: a database of human splicing factors expression data and RNA target motifs
Last update: 2013 February:	71 splicing factors, 2,339 RNA binding sites

Organism: human
* [URL](http://193.206.120.249/splicing_tissue.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21922594/)

### TassDB: a tandem splice sites database
Last update: _not found_. Published in 2010.

* [URL 1](http://www.tassdb.info/)
* [URL 2](http://tassdb2.leibniz-fli.de/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20429909)
