# Alternative-Splicing-List
A list of alternative splicing analysis resources
## Splice-aware mapping tools
#### BFAST: a BLAT-like mapping tool
Last release: 2011, October, 29th
* [URL](https://sourceforge.net/projects/bfast/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19907642/)

#### GEM-mapper: a maping tool
Last release: 2020, September, 25th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23103880/)
* [GitHub](https://github.com/smarco/gem3-mapper)

#### GSNAP: a splice-aware mapping tool
Last release: 2021, February, 22nd
* [URL](http://research-pub.gene.com/gmap/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20147302/)

#### MapSplice3: a splice-aware mapping tool
Last release: 2019, May, 30th (the last commit)
* [URL](http://www.netlab.uky.edu/p/bioinfo/MapSplice2)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20802226/)
* [GitHub](https://github.com/LiuBioinfo/MapSplice)

#### PALMapper: a splice-aware mapping tool
Last release: 2013, July, 2013
* [URL](https://bmi.inf.ethz.ch/supplements/palmapper)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21154708/)

#### Rail-RNA: a cloud-enabled splice-aware mapping tool
Last release: 
* [URL](http://rail.bio/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27592709/)

#### RUM: a splice-aware mapping tool
Last release: 2015, January, 27th
* [URL](http://cbil.upenn.edu/RUM/)
* [GitHub](https://github.com/itmat/rum)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21775302/)

#### Scipio: a BLAT-based alignment tool to determine the gene structure given a protein sequence and a genome sequence
Last release: 2013, May, 24th
* [URL](https://www.webscipio.org/webscipio/download_scipio)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/18554390/)

#### SeqSaw: a splice-aware mapping tool
Last release:_ not found_
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/seqsaw/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21575597/)

#### SpliceMap: a splice-aware mapping tool
Last release: 2010, October, 23rd
* [URL](https://web.stanford.edu/group/wonglab/SpliceMap/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20371516/)

#### Subread: a splice-aware mapping tool
Last release: 2020, May, 13th

featureCounts is a part of Subread to count features expression
* [URL](http://subread.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30783653/)

#### STAR: a splice-aware mapping tool
Last release: 2021, February, 20th
* [GitHub](https://github.com/alexdobin/STAR/releases)
* [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/)

#### UnSplicer: a splice-aware alignment and junction detection tool
Last release: 2013, November, 25th
* [URL](http://exon.gatech.edu/Genemark/paul/unsplicer/index.htm)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24259430/)

## Alternative splicing and differential splicing detection tools
### Event-level
### AltAnalyze: an alternative splicing event detection tool
Kallisto-Splice is a part of a tool for a splice-aware mapping

Last release: 2021, January, 14th
* [URL](http://www.altanalyze.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20513647/)

#### ASprofile: an alternative splicing event detection tool
Last release: 
* [URL](http://ccb.jhu.edu/software/ASprofile/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24555089/)

#### casper: an alternative splicing event detection tool
Last release:_ not found_. Published in 2014.

* [Biocondictor](https://www.bioconductor.org/packages/release/bioc/html/casper.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24795787/)

#### DSGseq: a differential splicing detection tool
Last release: 2012, May, 14th
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/DSGseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23228854/)

#### EventPointer: an alternative splicing event detection tool
Last release: *not found* 
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27315794/)
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/EventPointer.html)

#### GESS: an exon skipping detection tool
Last release: *not found*
* [URL](http://jinlab.net/GESS_Web/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24369421/)

#### jSplice: a differential splicing detection tool
Last release: 2017, July, 4th. Not maintained.
* [URL]()
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27153587/)
* [GitHub](https://github.com/yannchristinat/jsplice)

#### KISSPLICE: an alternative splicing event detection tool
Last release: 2020, July, 2nd
* [URL](http://kissplice.prabi.fr/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22537044/)

#### MAJIQ: an alternative splicing event detection tool
Last release: *not found*
* [URL](https://majiq.biociphers.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26829591/)

### nagnag: a nagnag splicing prediction tool
Last release: 2015, August, 4th
* [URL](https://sourceforge.net/projects/nagnag/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26028313/)

### PTESFinder: a post-transcriptional exon shuffling
Last release: 2019, September, 4th
* [URL](https://sourceforge.net/projects/ptesfinder-v1/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26758031/)

#### rMATS: a differential splicing detection
Last release: 2021, February, 8th
* [URL](http://rnaseq-mats.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25480548/)
* [GitHub](https://github.com/Xinglab/rmats-turbo)

#### RNAprof: a differential RNA processing tool
Last release: 2017, May, 24th
* [URL](http://rssf.i2bc.paris-saclay.fr/Software/rnaprof.php)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26849165/)

#### SeqGSEA: a gene set enrichment analysis with differential expression and splicing
Last release: 2018, December, 12th (the last commit)

* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SeqGSEA.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734663/)
* [GitHub](https://github.com/sunlightwang/SeqGSEA_1.13)

#### SGSeq: an alternative splicing event detection tool
Last release:*not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27218464/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SGSeq.html)

#### splAdder: an alternative splicing event detection tool
Last release: 2019, November, 19th
* [URL](https://bmi.inf.ethz.ch/supplements/spladder)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26873928/)
* [GitHub](https://github.com/ratschlab/spladder)

#### SplicePie: an order of splicing prediction
Last release: 2015, February, 20th (the last commit)

* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25800735/)
* [GitHub](https://github.com/pulyakhina/splicing_analysis_pipeline)

#### spliceR: a tool for alternative splicing events, differential splicing detection and isoform switching
Last release:_ not found_. Published in 2014

* [Bioconductor](http://www.bioconductor.org/packages/2.13/bioc/html/spliceR.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24655717/)

#### SpliceSeq: a tool for investigation of alternative mRNA splicing patterns in data from RNA-seq.
The tool detects events, compares between samples and predicts the consequence on a protein structure

Last release: 2014, February, 21st
* [URL](https://bioinformatics.mdanderson.org/public-software/spliceseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820202/)

#### SplicingCompass: a differential splicing tool
Last release: 2019, October, 24th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23449093/)
* [GitHub](https://github.com/KoenigLabNM/SplicingCompass)

#### SplicingTypesAnno: a alternative splicing event detection tool
Last release: 2015, July, 31st
* [URL](https://sourceforge.net/projects/splicingtypes/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25720307/)

#### Vials: a visualization of alternative splicing tool
Last release: 2017, February, 7th
* [URL](http://vials.io/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26529712/)
* [GitHub (archived)](https://github.com/Caleydo/vials)

#### Webscipio: a web tool for prediction of eukaryotic gene structures
Last release:
* [URL](https://www.webscipio.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21718515/)

### Isoform-level
#### Alexa-Seq: an isoform qunification and differential isoform expression tool
Last release: *not found*. Not maintained.
* [URL](http://www.alexaplatform.org/alexa_seq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20835245/)

#### Augustus: an isoform inference tool
Last release: 2020, December, 11th
* [URL](http://bioinf.uni-greifswald.de/augustus/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16845043/)
* [GitHub](https://github.com/Gaius-Augustus/Augustus)

#### BitSeq: an isoform quantification and differential isoform expression tool
Last release: *not found*
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/BitSeq.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22563066/)

#### CEM: a transcriptome assmbly and isoform quantification tool
Last release: 2012, November, 17th
* [URL](http://alumni.cs.ucr.edu/~liw/cem.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23060617/)

#### CLASS2: a transcriptome assembly tool
Last release: 2017, May, 17th
* [URL](https://sourceforge.net/projects/splicebox/files/)
* [PubMed CLASS](https://pubmed.ncbi.nlm.nih.gov/23734605/)
* [PubMed CLASS2](https://pubmed.ncbi.nlm.nih.gov/26975657/)

#### Cufflinks: a suite of tools for an isoform quantification and differential isoform expression tool
* Cufflinks assembles transcriptomes from RNA-Seq data and quantifies their expression.
* Cuffcompare compares the assembled transcriptomes
* Cuffmerge merges the tranacriptomes from multiple RNA-Seq libraries
* Cuffquant computes the gene and transcript expression profiles 
* Cuffdiff compares expression levels of genes and transcripts in RNA-Seq
* Cuffnorm normalizes the expression levels
Last release: 2016, July, 1st
* [URL](http://cole-trapnell-lab.github.io/cufflinks/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23222703/)
* [GitHub](https://github.com/cole-trapnell-lab/cufflinks)

#### DiffSplice: a differential isoform expression detection tool
Last release: 2013, January, 1st
* [URL](http://www.netlab.uky.edu/p/bioinfo/DiffSplice)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23155066/)

#### DRIMseq: a differential isoform expression detection tool
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28105305/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/DRIMSeq.html)

#### EBseq: a differential isoform expression detection tool
Last release:_ not found_
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23428641/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/EBSeq.html)

#### FineSplice: a TopHat2 wrapper to identify expressed juntions
Last release: 2014, April, 1st
* [URL](https://sourceforge.net/projects/finesplice/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24574529/)

#### Integrated Genome Browser: a visual analysis of alternative splicing
Last release: 2020, December, 3rd 
* [URL](https://www.bioviz.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24792048/)

#### iReckon: an isoform inference and quantification tool
Last release: *not found*
* [URL](http://compbio.cs.toronto.edu/ireckon/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23204306/)

#### IsoEm2: an isoform quantification tool
Last release: 2017, April, 3rd
* [URL](https://dna.engr.uconn.edu/?page_id=105)
* [GitHub](https://github.com/mandricigor/isoem2)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21504602/)

#### IsoInfer: a tool for isoforms inference
Last release: 2010, September, 9th
* [URL](http://www.cs.ucr.edu/%E2%88%BCjianxing/IsoInfer.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21385036/)

#### iso-kTSP: an isoform switch detection
Last release: 2015, May, 5th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25578962/)
* [BitBucket](https://bitbucket.org/regulatorygenomicsupf/iso-ktsp/src/master/)

#### isoLASSO: a transcriptome assembly and isoform quantification tool
Last release: 2012, November, 17th
* [URL](http://alumni.cs.ucr.edu/~liw/isolasso.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21951053/)

#### ISP: a tool for isoforms inference
Last release: 2014, October, 15th
* [URL](http://alumni.cs.ucr.edu/~liw/isp.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25708199/)

#### IUTA: a differential isoform expression tool
Last release:_ not found_. Published in 2014.
* [URL](https://www.niehs.nih.gov/research/resources/software/biostatistics/iuta/index.cfm)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25283306/)

#### kallisto: an isoform quantification tool
Last release: 2020, February, 13th
* [URL](https://pachterlab.github.io/kallisto/about)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27043002/)
* [GitHub](https://github.com/pachterlab/kallisto)

#### MMSEQ: an isoform quantification tool
Last release: 2019, November, 11th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21310039/)
* [GitHub](https://github.com/eturro/mmseq)

#### MISO: a differential isoform expression tool
Last release: 2019, May, 17th. No longer maintained.
* [URL](http://hollywood.mit.edu/burgelab/miso/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21057496/)
* [GitHub](https://github.com/yarden/MISO)

#### MIETIE: an isoform inference and quantification tool
Last release: 2016, March, 18th
* [URL](https://bmi.inf.ethz.ch/supplements/mitie)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23980025/)
* [GitHub](https://github.com/ratschlab/MiTie)

#### mtim: an isoform inference tool
Last release: 2013, April, 18th
* [arXiv](https://arxiv.org/abs/1309.5211)
* [GitHub](https://github.com/nicococo/mTIM)

#### NLDMSeq: an isoform quantification tool
Last release: 2016, June, 27th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26475308/)
* [GitHub](https://github.com/PUGEA/NLDMseq)

#### NURD: an isoform quantification tool
Last release: _not found_. Published in 2013.
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/NURD/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23837734/)

#### Oases: a transcriptome assembly tool
Last release: 2011, December, 6th
* [URL](https://www.ebi.ac.uk/~zerbino/oases/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22368243/)

#### PSGInfer: an isoforms inference and differential analysis tool
Last release: 2013, October, 25th
* [URL](http://deweylab.biostat.wisc.edu/psginfer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23846746/)

#### RNASeqViewer: a visualization tool for gene expression and isoforms
Last release: 2013, October, 14th
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/RNAseqViewer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24215023/)

#### rDiff: a differential isoform expression detection tool
Last release: 2020, June, 19th (the last commit)
* [URL](https://public.bmi.inf.ethz.ch/user/drewe/rdiff/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23585274/)
* [GitHub](https://github.com/ratschlab/rDiff)

#### RSEM: an isoform quantification tool
The variant of the tools is pRSEM: an isoform quantification from a complementary data
Last release: 2020, February, 14th
* [URL RSEM](https://deweylab.github.io/RSEM/)
* [URL pRSEM](https://deweylab.github.io/pRSEM/)
* [PubMed RSEM](https://pubmed.ncbi.nlm.nih.gov/21816040/)
* [PubMed pRSEM](https://pubmed.ncbi.nlm.nih.gov/27405803/)
* [GitHub](https://github.com/deweylab/RSEM)

#### rSeqDiff: a differential isoform expression detection tool
Last release: _not found_. Published in 2013
* [URL](https://jhui2014.github.io/rseqdiff/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24260225/)

#### SDEAP: a differential isoform expression detection tool for population data
Last release: 2017, March, 15th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27522083/)
* [GitHub](https://github.com/ewyang089/SDEAP/wiki)

#### Scripture: a transcriptome assembly tool
Last release:
* [URL](http://software.broadinstitute.org/software/scripture/home)
* [PubMed]()

#### SigFuge: a differential isoform expression detection tool
Last release:_ not found_. Published in 2014.
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/SigFuge.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25030904/)

#### SLIDE: an isoform inference and quantification tool
Last release: 2012, May, 17th
* [URL](http://www.stat.ucla.edu/~jingyi.li/software-and-data.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22135461/)

#### SOAPdenovo-trans: a transcriptome assembly tool
Last release: 2017, August, 23rd
* [URL](https://sourceforge.net/projects/soapdenovotrans/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24532719/)
* [GitHub](https://github.com/aquaskyline/SOAPdenovo-Trans)

#### SpliceGrapher: a tool for creating splice graphs and isoforms predicting
Last release: 2014, April, 24th
* [URL](http://splicegrapher.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22293517/)

#### spliceR: a tool for alternative splicing, differential detection and isoform switching
Last release:_ not found_. Published in 2014

* [Bioconductor](http://www.bioconductor.org/packages/2.13/bioc/html/spliceR.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24655717/)

#### StringTie: a transcriptome assembly and isoform quantification tool
Last release: 2021, March, 4th
* [URL](http://ccb.jhu.edu/software/stringtie/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25690850/)

#### SUPPA: an isoform quantification tool
Last release: 2018, February, 22nd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26179515/)
* [GitHub](https://github.com/comprna/SUPPA)
* [BitBucker](https://bitbucket.org/regulatorygenomicsupf/suppa/src/master/)

#### TIGAR2: an isoform quantification tool from longer RNA-Seq
Last release: 2015, February, 19th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23821651/)
* [GitHub](https://github.com/nariai/tigar2)

#### TRANS-Abyss: a transcriptome assembly tool
Last release: 2018, February, 19th
* [URL](https://www.bcgsc.ca/resources/software/trans-abyss)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20935650/)
* [GitHub](https://github.com/bcgsc/transabyss)

#### Traph: an isoform quantification tool
Last release: 2014, September, 2nd
* [URL](https://www.cs.helsinki.fi/en/gsa/traph/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734627/)

#### TRIMD: Iso-Seq, RNA-Seq and deepCAGE data integration for transcriptome resolution (Perl scripts)
Last release: 2020, September, 9th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27407110/)
* [GitHub](https://github.com/flemingtonlab/TRIMD)

#### Trinity: a transcriptome assembly tool
Last release: 2015, January, 24th
* [URL](https://sourceforge.net/projects/trinityrnaseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21572440/)

#### Velvet: a transcriptome assembly tool
Last release: 2018, July, 5th
* [URL](https://www.ebi.ac.uk/~zerbino/velvet/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/18349386/)
* [GitHub](https://github.com/dzerbino/velvet/tree/master)

### Exon-level
#### DEXSeq: a differential exon usage tool
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/DEXSeq.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722343/)

#### ExClust: a package for constitutive exons analysis
Part of the sysSeq package.

Last release: 2015, July, 17th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23360225/)
* [GitHub](https://github.com/ellispatrick/sydSeq)

#### ExCluster: a differential exon expression tool
Last release: _not found_. Published in 2020.
* [BioConductor](https://bioconductor.org/packages/release/bioc/html/ExCluster.html)

#### HMMSplicer: a splice junctions detection
Last release: 2010, November, 25th
* [URL](http://derisilab.ucsf.edu/index.php?software=105)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21079731/)

#### HTSeq: a RNA-Seq quantification tool
The tool can quantify exon expression

Last release: 2020, April, 18th
* [URL](https://htseq.readthedocs.io/en/master/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25260700/)
* [GitHub](https://github.com/htseq/htseq)

#### iGEMS: a model for alternative exon usage detection (R scripts)
Last release: 2016, April, 13th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27095197/)
* [GitHub](https://github.com/iaingallagher/iGEMS_scripts)

#### JunctionSeq: a differential exon and junctions usage
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27257077/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/JunctionSeq.html)

#### MISO: a differential isoform expression tool
The tool also detects differentially regulated exons.
MISO also presents sashimi plots for exon expression visualisation.

Last release: 2019, May, 17th. No longer maintained.
* [URL](http://hollywood.mit.edu/burgelab/miso/)
* [PubMed MISO](https://pubmed.ncbi.nlm.nih.gov/21057496/)
* [PubMed sashimi plots](https://pubmed.ncbi.nlm.nih.gov/25617416/)
* [GitHub](https://github.com/yarden/MISO)

#### Read-Split-Run: a splice junctions detection
Last release: 2016, January, 6th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27556805/)
* [GitHub](https://github.com/xuric/read-split-run)

#### SpliceJumper: a splice junctions detection
Last release: 2015, June, 29th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26678515/)
* [GitHub](https://github.com/simoncchu/SpliceJumper)

#### SpliceTrap: a tool to quantify exon inclusion levels using paired-end RNA-seq data
Last release: 2013, May, 17th
* [URL](http://rulai.cshl.edu/splicetrap/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21896509/)

#### SplicingGraphs: a splicing graphs creation, visualization and quantification tool
Last release: 2020, November, 3rd
* [GitHub](https://github.com/Bioconductor/SplicingGraphs)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SplicingGraphs.html)

## sQTL analysis
### GLiMMPS: an sQTL detection
Last release: 2013, March, 12th
* [GutHub](https://github.com/Xinglab/GLiMMPS)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23876401/)

### PVAAS: a tool to identify SNVs associated with aberrant splicing
Last release:
* [URL](http://pvaas.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25573917/)

### sQTLseekeR: an sQTL detection
Last release: 2015, June, 29th
* [GitHub](https://github.com/guigolab/sQTLseekeR)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25140736/)

### SplicePlot: an sQTL visualization
Last release: 2013, October, 15th
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/24363378/)
* [GitHub](https://github.com/wueric/SplicePlot)

## Splice sites prediction and variations analysis
### EBChangePoint: a splice site prediction
Last release: 2015, May
* [URL](http://ebchangepoint.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26873932/)

### GeneSplicer: a splice site prediction
Last release:_ not found_
* [URL](https://www.cbcb.umd.edu/software/GeneSplicer/gene_spl.shtml)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/11222768/)

### Human Splicing Finder: a splice site prediction (free limited access)
Last release:_ not found_
* [URL](https://www.genomnis.com/access-hsf)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19339519/)

### MaxEntScan: a tool for human 5'splice sites scoring
Last release:_ not found_
* [URL](http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html)

### NetAspGene: a splice site prediction in Aspergillus
Last release:_ not found_
* [URL](https://services.healthtech.dtu.dk/service.php?NetAspGene-1.0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/18948220/)

### NetGene2 Server: a splice site prediction
Last release: 2017, May, 30th
* [URL 1](http://www.cbs.dtu.dk/services/NetGene2/)
* [URL 2](https://services.healthtech.dtu.dk/service.php?NetGene2-2.42)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/2067018/)

### NetUTR: a splice site presiction in 5'UTR regions of human genes
Last release: _ not found_
* [URL](https://services.healthtech.dtu.dk/service.php?NetUTR-1.0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/14960723/)

### NNSPLICE: a splice site prediction by neural network
Last release: 1997, January
* [URL](https://www.fruitfly.org/seq_tools/splice.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/9278062/)

### SpliceView: a splice site prediction
Last release:_ not found_
* [URL](http://bioinfo.itb.cnr.it/~webgene/wwwspliceview.html)

### SPANR: a splicing-based analysis of SNVs
Last release: 
* [URL](http://tools.genes.toronto.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25525159/)

## Alternative splicing regulation
### ExonSuite: a tool for PUF binding regions analysis
Last release: 2013, January, 13th (the last commit)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/23816174/)
* [GitHub](https://github.com/dilanustek/ExonSuite)

## Proteomics
### QUILTS: the identification of novel proteins, resulting from single nucleotide variants, splice variants and fusion genes
Last release: *not found*
* [URL](http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26631509/)
* [GitHub](https://github.com/ekawaler/pyQUILTS/) - on a very early stage

### SpliceVista: a tool for splice variant identification and visualization in shotgun proteomics data
Last release: 2017, June 21st (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24692640/)
* [GitHub](https://github.com/yafeng/SpliceVista)

## Network analysis
### CoSpliceNet: a co-splicing network tool
Last release: 2017, October, 18th (the last commit)
* [URL](http://delasa.github.io/co-spliceNet/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27793091/)
* [GitHub](https://github.com/Delasa/co-spliceNet)

### DomainGraph: effect of alternative splicing on domain-domain interactions
Last release: December, 2018
* [URL](https://domaingraph.bioinf.mpi-inf.mpg.de/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20513647/)

## Single cell data
### SingleSplice
Last release: 2016, July, 7th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26740580/)
* [GitHub](https://github.com/jw156605/SingleSplice)

## Other
### BEERS: RNA-Seq reads with splicing simulation
Last release: 2015, December, 9th (the last commit)
* [URL](http://cbil.upenn.edu/BEERS/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21775302/)
* [GitHub](https://github.com/itmat/beers_simulator)

### FSOM: a methylation patterns associated with alternative splicing
Last release: 2015, September, 3rd
* [URL](https://sourceforge.net/projects/fsom/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26365234/)

### PPLine: a pipeline for SNP calling and gene/isoform quantification
Last release: 2018, March, 26th
* [URL](https://sourceforge.net/projects/ppline/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26147802/)

### PrimerSeq: a tool for RT-PCR primers design that evaluate alternative splicing events
Last release: 2016, November, 18th
* [URL](http://primerseq.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24747190/)
* [GitHub](https://github.com/ctokheim/PrimerSeq)

## Databases
### AceView: a database of alternative splicing isoforms
Last update: 2012, October, 16th

Organisms: human, mouse, rat, worm, arabidopsis
* [URL](https://www.ncbi.nlm.nih.gov/IEB/Research/Acembly/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16925834/)

### AltTrans: a database of alternative splicing and alternative polyadenilation (archived)
Last update: 2008, September, 5th

Organisms: human, mouse, rat
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16556303/)
* [FTP](ftp://ftp.ebi.ac.uk/pub/databases/astd)

### APPRIS: a database of principal splice isoforms
Last update: 2021

Organisims: 19 vertebrate and invertebrate genomes
* [URL](http://appris.bioinfo.cnio.es/#/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23161672)

### ASPicDB: a database of alternative splicing patterns in human genes
Last update: 2017, May, 4th, 196,446 splicing events	

Organism: human
* [URL](http://srv00.recas.ba.infn.it/ASPicDB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21051348/)

### DBTSS: a database of transcription start sites
Last update: 2017, December, 21st

Organism: human
* [URL](https://dbtss.hgc.jp/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/29126224)

### H-DBAS: a human alternative splicing database
Last update: 2011, December, 26th

Organism: human
* [URL](http://h-invitational.jp/h-dbas/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/19969536?dopt=Abstract)

### HEXEvent: a database of Human EXon splicing Events
Last update: 2017, September, 29th

Organism: human
* [URL](http://hexevent.mmg.uci.edu/cgi-bin/HEXEvent/HEXEventWEB.cgi)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23118488/)

### ISOexpresso: a database of isoform expression in cancer
Last update: 2016, June, 1st

Organism: human
* [URL](http://wiki.tgilab.org/ISOexpresso/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27519173/)

### Kassiopeia: a database of mutually exclusive exonomes
Last update: _not found_

Organism: human, fruit fly, worm, Arabidopsis
* [URL](https://kassiopeia.motorprotein.de/kassiopeia)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24507667/)

### PolyA_DB: a database of pre-mRNA cleavage and polyA sites
Last update: 2018, August. 107 human samples, 246 mouse samples, 11 rat samples, 9 chicken samples.

Organisms: human, mouse, rat, chicken
* [URL](https://exon.apps.wistar.org/PolyA_DB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29069441/)

### ProSAS: a database of the effects of alternative splicing events on the structure of the resulting protein isoforms.
Last update: _not found_. Published in 2008.

* [URL](https://drupal.bio.ifi.lmu.de/forschung/structural-bioinformatics/prosas/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/17933774)

### SpliceAid2: a database of human splicing factors expression data and RNA target motifs
Last update: 2013 February:	71 splicing factors, 2,339 RNA binding sites

Organism: human
* [URL](http://193.206.120.249/splicing_tissue.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21922594/)

### TassDB: a tandem splice sites database
Last update: _not found_. Published in 2010.

* [URL 1](http://www.tassdb.info/)
* [URL 2](http://tassdb2.leibniz-fli.de/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20429909)

### TCGA SpliceSeq: splicing patterns alternation in TCGA
Last update: *not found*
* [URL](https://bioinformatics.mdanderson.org/TCGASpliceSeq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26602693/)
