# Alternative Splicing Tools
A list of alternative splicing analysis resources
## Splice-aware mapping tools
#### BBMap
Last release: 2021, February, 3rd
* [URL](https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/)
* [GitHub](https://github.com/BioInfoTools/BBMap)

#### BFAST: a BLAT-like mapping tool
Last release: 2011, October, 29th
* [URL](https://sourceforge.net/projects/bfast/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19907642/)

#### ContextMap2
Last release: *not found*
* [URL](https://www.bio.ifi.lmu.de/software/contextmap/)
* [PubMed](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0557-5)

#### CRAC
Last release: 2016, January, 29th
* [URL](http://crac.gforge.inria.fr/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23537109/)

#### DART
Last release: 2019, May, 7th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28968831/)
* [GitHub](https://github.com/hsinnan75/Dart)

#### GEM-mapper
Last release: 2020, September, 25th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23103880/)
* [GitHub](https://github.com/smarco/gem3-mapper)

#### GSNAP
Last release: 2021, February, 22nd
* [URL](http://research-pub.gene.com/gmap/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20147302/)

#### HISAT2
Last release: 2020, July, 24th
* [URL](http://daehwankimlab.github.io/hisat2/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31375807/)
* [GitHub](https://github.com/DaehwanKimLab/hisat2)

#### iMapSplice: a personalized RNA-Seq alignment tool
Last release: 2019, February, 8th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30096157/)
* [GitHub](https://github.com/LiuBioinfo/iMapSplice)

#### Magic-BLAST
Last release: 2021, March
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31345161/)
* [GitHub](https://github.com/ncbi/magicblast)

#### MapSplice3
Last release: 2019, May, 30th (the last commit)
* [URL](http://www.netlab.uky.edu/p/bioinfo/MapSplice2)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20802226/)
* [GitHub](https://github.com/LiuBioinfo/MapSplice)

#### minimap2
Last release: 2019, May, 5th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29750242/)
* [GitHub](https://github.com/lh3/minimap2)

#### PALMapper: a splice-aware mapping tool
Last release: 2013, July, 2013
* [URL](https://bmi.inf.ethz.ch/supplements/palmapper)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21154708/)

#### Rail-RNA: a cloud-enabled splice-aware mapping tool
Last release: 
* [URL](http://rail.bio/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27592709/)

#### RUM: a splice-aware mapping tool
Last release: 2015, January, 27th
* [URL](http://cbil.upenn.edu/RUM/)
* [GitHub](https://github.com/itmat/rum)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21775302/)

#### Scipio: a BLAT-based alignment tool to determine the gene structure given a protein sequence and a genome sequence
Last release: 2013, May, 24th
* [URL](https://www.webscipio.org/webscipio/download_scipio)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/18554390/)

#### segemehl
Last release: 2018, December, 25th
* [URL](https://www.bioinf.uni-leipzig.de/Software/segemehl/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24626854/)

#### SeqSaw: a splice-aware mapping tool
Last release:_ not found_
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/seqsaw/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21575597/)

#### Splice2Deep: a splice site prediction
Last release: 2019, August, 7th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32550561/)
* [GitHub](https://github.com/SomayahAlbaradei/Splice_Deep)

#### SpliceFamAlign: a CDS-to-gene alignment
Last release: 2018, August, 6th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30925859/)
* [GitHub](https://github.com/UdeS-CoBIUS/SpliceFamAlign)

#### SpliceMap: a splice-aware mapping tool
Last release: 2010, October, 23rd
* [URL](https://web.stanford.edu/group/wonglab/SpliceMap/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20371516/)

#### Subread: a splice-aware mapping tool
Last release: 2020, May, 13th

featureCounts is a part of Subread to count features expression
* [URL](http://subread.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30783653/)

#### STAR: a splice-aware mapping tool
Last release: 2021, February, 20th
* [GitHub](https://github.com/alexdobin/STAR/releases)
* [Pubmed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3530905/)

#### UnSplicer: a splice-aware alignment and junction detection tool
Last release: 2013, November, 25th
* [URL](http://exon.gatech.edu/Genemark/paul/unsplicer/index.htm)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24259430/)

## Alternative splicing and differential splicing detection tools
### Event-level
### AltAnalyze: an alternative splicing event detection tool
Kallisto-Splice is a part of a tool for a splice-aware mapping

Last release: 2021, January, 14th
* [URL](http://www.altanalyze.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20513647/)

### ARH-Seq: a differential splicing detection tool
Last release: 2014

The R code is in the supplementary materials
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24920826/)

#### ASDT: an alternative splicing event detection tool
Last release: 2018, April, 17th
* [URL](http://aias.biol.uoa.gr/~mtheo/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30069446/)

#### ASGAL: an alternative splicing event detection tool
Last release: 2020, June, 22nd
* [URL](https://asgal.algolab.eu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30458725/)
* [GitHub](https://github.com/AlgoLab/galig)

#### ASpli: an alternative splicing event and differential splicing detection tool
Last release: 2017, February, 25th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33677494/)
* [GitHub](https://github.com/estepi/ASpli)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/ASpli.html)

#### ASprofile: an alternative splicing event detection tool
Last release: 
* [URL](http://ccb.jhu.edu/software/ASprofile/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24555089/)

#### AStrap: an alternative splicing event prediction from transcript sequence
Last release: 2020, November, 13th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30535139/)
* [GitHub](https://github.com/BMILAB/AStrap)

#### CASH: a differential splicing analysis tool
Last release: 2018, April, 3rd
* [URL 1](https://soft.novelbio.com/cash/)
* [URL 2](https://sourceforge.net/projects/cash-program/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28387786/)

#### CATANA: an alternative splicing annotation
Last release: 2018, November, 28th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30202999/)
* [GitHub](https://github.com/shiauck/CATANA)

#### casper: an alternative splicing event detection tool
Last release: *not found*
* [Biocondictor](https://www.bioconductor.org/packages/release/bioc/html/casper.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24795787/)

#### COSSMO: an alternative splicing site usage prediction
Last release: 2018, July, 4th (the last commit)
* [URL](http://cossmo.genes.toronto.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29949959/)
* [GitHub](https://github.com/PSI-Lab/COSSMO)

#### DARTS: an alternative splicing event detection tool
Last release: 2019, February, 26th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30923373/)
* [GitHub](https://github.com/Xinglab/DARTS)

#### Deep Splicing Code: an alternative splicing event prediction from sequence
Last release: 2020, September, 5th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31374967/)
* [GitHub](https://github.com/louadi/DSC)

#### DeepDSSR: a donor splice site recognition (a model)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31349311/)

#### DIEGO: a differential alternative splicing tool
Last release: 2019, December, 3rd
* [URL](http://www.bioinf.uni-leipzig.de/Software/DIEGO/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29088309/)

#### DSGseq: a differential splicing detection tool
Last release: 2012, May, 14th
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/DSGseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23228854/)

#### EventPointer: an alternative splicing event detection tool
Last release: *not found* 
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/27315794/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/30253752/)
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/EventPointer.html)

#### FakIR: an intron retention identification tool
Last release: 2021, January, 28th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33073477/)
* [GitHub](https://github.com/lbroseus/FakIR)

#### FARLINE: a pipeline for alternative splicing event detection
Last release: *not found*
* [URL](http://kissplice.prabi.fr/pipeline_ks_farline/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29523794/)

#### GESS: an exon skipping detection tool
Last release: *not found*
* [URL](http://jinlab.net/GESS_Web/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24369421/)

#### iDiffIR: a differential intron retention tool
Last release: 2015
* [URL](https://combi.cs.colostate.edu/idiffir/introduction.html)
* [Bitbucket](https://bitbucket.org/comp_bio/idiffir/src/master/)

#### IntEREst: an intron retention identification tool
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29642843/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/IntEREst.html)

#### iREAD: an intron retention identification tool
Last release: 2021, February, 16th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32028886/)
* [GitHub](https://github.com/genemine/iread)

#### IRFinder: an intron retention identification tool
Last release: 2020, August, 27th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28298237/)
* [GitHub](https://github.com/williamritchie/IRFinder)

#### jSplice: a differential splicing detection tool
Last release: 2017, July, 4th. Not maintained.
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27153587/)
* [GitHub](https://github.com/yannchristinat/jsplice)

#### JUM: an alternative splicing event detection tool
Last release: 2019, July, 2019
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30104386/)
* [GitHub](https://github.com/qqwang-berkeley/JUM)

#### Keep Me Around: an intron retention identification tool
Last release: 2019, January, 30th (the last commit)
* [arRxiv](https://arxiv.org/abs/1510.00696)
* [GitHub](https://github.com/pachterlab/kma)

#### KISSPLICE: an alternative splicing event detection tool
KISSDE for differential splicing analysis
Last release: 2020, July, 2nd
* [URL 1](http://kissplice.prabi.fr/)
* [URL 2](http://kissplice.prabi.fr/pipeline_ks_farline/)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/22537044/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/29523794/)

#### MAJIQ: an alternative splicing event detection tool
Last release: *not found*
* [URL](https://majiq.biociphers.org/)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/26829591/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/29236961/)

#### Manananggal: a visualization of alternative splicing events
Last release: 2016, December, 8th (the last commit)
* [PubMed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5319012/)
* [GitHub](https://github.com/barannm/Manananggal)

#### nagnag: a nagnag splicing prediction tool
Last release: 2015, August, 4th
* [URL](https://sourceforge.net/projects/nagnag/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26028313/)

#### psichomics: an alternative splicing quantification and visualization tool
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30277515/)
* [Bioconductor](http://bioconductor.org/packages/release/bioc/html/psichomics.html)

#### PSI-Sigma: a differential splicing detection tool
Last release: 2021, February, 16th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31135034/)
* [GitHub](https://github.com/wososa/PSI-Sigma)

#### PTESFinder: a post-transcriptional exon shuffling
Last release: 2019, September, 4th
* [URL](https://sourceforge.net/projects/ptesfinder-v1/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26758031/)

#### rMATS: a differential splicing detection
Last release: 2021, February, 8th
* [URL](http://rnaseq-mats.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25480548/)
* [GitHub](https://github.com/Xinglab/rmats-turbo)

#### RNAprof: a differential RNA processing tool
Last release: 2017, May, 24th
* [URL](http://rssf.i2bc.paris-saclay.fr/Software/rnaprof.php)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26849165/)

#### SAJR: a differential splicing tool
Last release: *not found*
* [URL](http://storage.bioinf.fbb.msu.ru/~mazin/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23340839/)

#### SeqGSEA: a gene set enrichment analysis with differential expression and splicing
Last release: 2018, December, 12th (the last commit)

* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SeqGSEA.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734663/)
* [GitHub](https://github.com/sunlightwang/SeqGSEA_1.13)

#### SGSeq: an alternative splicing event detection tool
Last release:*not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27218464/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SGSeq.html)

#### splAdder: an alternative splicing event detection tool
Last release: 2019, November, 19th
* [URL](https://bmi.inf.ethz.ch/supplements/spladder)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26873928/)
* [GitHub](https://github.com/ratschlab/spladder)

#### SpliceDetector: an alternative splicing event detection tool
Last release: *not found*
* [URL Google Drive](https://drive.google.com/open?id=1dlXKzbvxOH3A85_DVR__V2eI5s16-llv)
* [URL Dropbox](https://www.dropbox.com/s/j5o0og159ig6tej/SpliceDetector%20Executable%20File.rar?dl=0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29567976/)

#### SplicePie: an order of splicing prediction
Last release: 2015, February, 20th (the last commit)

* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25800735/)
* [GitHub](https://github.com/pulyakhina/splicing_analysis_pipeline)

#### spliceR: a tool for alternative splicing events, differential splicing detection and isoform switching
Last release:_ not found_. Published in 2014

* [Bioconductor](http://www.bioconductor.org/packages/2.13/bioc/html/spliceR.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24655717/)

#### SpliceSeq: a tool for investigation of alternative mRNA splicing patterns in data from RNA-seq.
The tool detects events, compares between samples and predicts the consequence on a protein structure

Last release: 2014, February, 21st
* [URL](https://bioinformatics.mdanderson.org/public-software/spliceseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820202/)

#### SplicingCompass: a differential splicing tool
Last release: 2019, October, 24th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23449093/)
* [GitHub](https://github.com/KoenigLabNM/SplicingCompass)

#### SplicingTypesAnno: a alternative splicing event detection tool
Last release: 2015, July, 31st
* [URL](https://sourceforge.net/projects/splicingtypes/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25720307/)

#### Vials: a visualization of alternative splicing tool
Last release: 2017, February, 7th
* [URL](http://vials.io/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26529712/)
* [GitHub (archived)](https://github.com/Caleydo/vials)

#### Webscipio: a web tool for prediction of eukaryotic gene structures
Last release:
* [URL](https://www.webscipio.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21718515/)

#### Whippet: an alternative splicing event detection tool
Last release: 2021, March
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30220560/)
* [GitHub](https://github.com/timbitz/Whippet.jl)

#### Yanagi: a differential alternative splicing tool
Last release: 2019, November, 27th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31409274/)
* [GitHub](https://github.com/HCBravoLab/yanagi)

### Isoform-level
#### AERON: an isoform quantification and gene-fusion detection from long reads
Last release: 2020, November, 23rd (the last commit)
* [bioRxiv](https://www.biorxiv.org/content/10.1101/2020.01.27.921338v1)
* [GitHub](https://github.com/SchulzLab/Aeron)

#### Alexa-Seq: an isoform qunification and differential isoform expression tool
Last release: *not found*. Not maintained.
* [URL](http://www.alexaplatform.org/alexa_seq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20835245/)

#### AstaFunk: an alternatively spliced domains detection
Last release: *not found*
* [URL](http://confluence.sammeth.net/display/ASTA/3.4+-+Tool+ASTAFUNK)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29868795/)

#### Augustus: an isoform inference tool
Last release: 2020, December, 11th
* [URL](http://bioinf.uni-greifswald.de/augustus/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16845043/)
* [GitHub](https://github.com/Gaius-Augustus/Augustus)

#### BANDITS: a differential isoform usage
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32178699/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/BANDITS.html)

#### BIISQ: an isoform inference tool
Last release: 2017, March, 19th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29703885/)
* [GitHub](https://github.com/bee-hive/BIISQ)

#### BitSeq: an isoform quantification and differential isoform expression tool
Last release: 2019, August, 12th (the last commit cjBitSeq)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/BitSeq.html)
* [PubMed BitSeq](https://pubmed.ncbi.nlm.nih.gov/22563066/)
* [PubMed cjBitSeq](https://pubmed.ncbi.nlm.nih.gov/29091583/)
* [GitHub cjBitSeq](https://github.com/mqbssppe/cjBitSeq)

#### CAS-viewer: a visualization tool of cancer alternatively splicing
Last release: 2017
* [URL](http://genomics.chpc.utah.edu/cas/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29697367/)

#### CEM: a transcriptome assmbly and isoform quantification tool
Last release: 2012, November, 17th
* [URL](http://alumni.cs.ucr.edu/~liw/cem.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23060617/)

#### CIDANE: an isoform quantification and reconstruction tool
Last release: 2019, February, 12th
* [URL](http://ccb.jhu.edu/software/cidane/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30539557/)
* [Bitbucket](https://bitbucket.org/canzar/cidane/src/master/)

#### CLASS2: a transcriptome assembly tool
Last release: 2017, May, 17th
* [URL](https://sourceforge.net/projects/splicebox/files/)
* [PubMed CLASS](https://pubmed.ncbi.nlm.nih.gov/23734605/)
* [PubMed CLASS2](https://pubmed.ncbi.nlm.nih.gov/26975657/)

#### Cufflinks: a suite of tools for an isoform quantification and differential isoform expression tool
* Cufflinks assembles transcriptomes from RNA-Seq data and quantifies their expression.
* Cuffcompare compares the assembled transcriptomes
* Cuffmerge merges the tranacriptomes from multiple RNA-Seq libraries
* Cuffquant computes the gene and transcript expression profiles 
* Cuffdiff compares expression levels of genes and transcripts in RNA-Seq
* Cuffnorm normalizes the expression levels
Last release: 2016, July, 1st
* [URL](http://cole-trapnell-lab.github.io/cufflinks/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23222703/)
* [GitHub](https://github.com/cole-trapnell-lab/cufflinks)

#### DiffSplice: a differential isoform expression detection tool
Last release: 2013, January, 1st
* [URL](http://www.netlab.uky.edu/p/bioinfo/DiffSplice)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23155066/)

#### DRIMseq: a differential isoform expression detection tool
Last release: 2017, December, 20th (the last commit of BayesDRIMSeq)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28105305/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/DRIMSeq.html)
* [GitHub Bayes DRIMSeq](https://github.com/mqbssppe/BayesDRIMSeq)

#### EBseq: a differential isoform expression detection tool
Last release:_ not found_
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23428641/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/EBSeq.html)

#### Event Analysis: an isoform inference (a conda recipe)
Last release: 2019, April, 9th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30021829/)
* [GitHub](https://github.com/McIntyre-Lab/events)

#### FineSplice: a TopHat2 wrapper to identify expressed juntions
Last release: 2014, April, 1st
* [URL](https://sourceforge.net/projects/finesplice/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24574529/)

#### GFFView: a RNA-Seq viewer
Last release: *not found*
* [URL](http://120.27.116.110/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28355123/)

#### GSReg: a differential isoform expression
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29342249/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/GSReg.html)

#### HBA-DEALS: a differential gene and isoform expression
Last release: 2020, May, 11th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32660516/)
* [GitHub](https://github.com/TheJacksonLaboratory/HBA-DEALS)

#### Integrated Genome Browser: a visual analysis of alternative splicing
Last release: 2020, December, 3rd 
* [URL](https://www.bioviz.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24792048/)

#### iReckon: an isoform inference and quantification tool
Last release: *not found*
* [URL](http://compbio.cs.toronto.edu/ireckon/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23204306/)

#### IsoEm2: an isoform quantification tool
Last release: 2017, April, 3rd
* [URL](https://dna.engr.uconn.edu/?page_id=105)
* [GitHub](https://github.com/mandricigor/isoem2)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21504602/)

#### IsoformSwitchAnalyzer: an isoform switch detection
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30989184/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/IsoformSwitchAnalyzeR.html)

#### IsoInfer: a tool for isoforms inference
Last release: 2010, September, 9th
* [URL](http://www.cs.ucr.edu/%E2%88%BCjianxing/IsoInfer.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21385036/)

#### iso-kTSP: an isoform switch detection
Last release: 2015, May, 5th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25578962/)
* [BitBucket](https://bitbucket.org/regulatorygenomicsupf/iso-ktsp/src/master/)

#### isoLASSO: a transcriptome assembly and isoform quantification tool
Last release: 2012, November, 17th
* [URL](http://alumni.cs.ucr.edu/~liw/isolasso.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21951053/)

#### IsoSeq de novo: an isoform identification from IsoSeq (Python scripts)
Last release: 2017, September, 14th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28316149/)
* [GitHub](https://github.com/liuxiaoxian/IsoSeq_AS_de_novo)

#### ISP: a tool for isoforms inference
Last release: 2014, October, 15th
* [URL](http://alumni.cs.ucr.edu/~liw/isp.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25708199/)

#### IUTA: a differential isoform expression tool
Last release:_ not found_. Published in 2014.
* [URL](https://www.niehs.nih.gov/research/resources/software/biostatistics/iuta/index.cfm)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25283306/)

#### kallisto: an isoform quantification tool
sleuth is a differential isoform expression tool after kallisto analysis

Last release: 2020, February, 13th
* [URL kallisto](https://pachterlab.github.io/kallisto/about)
* [URL sleuth](http://pachterlab.github.io/sleuth/about)
* [PubMed kallisto](https://pubmed.ncbi.nlm.nih.gov/27043002/)
* [PubMed sleuth](https://pubmed.ncbi.nlm.nih.gov/28581496/)
* [GitHub kallisto](https://github.com/pachterlab/kallisto)

#### MMSEQ: an isoform quantification tool
Last release: 2019, November, 11th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21310039/)
* [GitHub](https://github.com/eturro/mmseq)

#### MISO: a differential isoform expression tool
Last release: 2019, May, 17th. No longer maintained.
* [URL](http://hollywood.mit.edu/burgelab/miso/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21057496/)
* [GitHub](https://github.com/yarden/MISO)

#### MIETIE: an isoform inference and quantification tool
Last release: 2016, March, 18th
* [URL](https://bmi.inf.ethz.ch/supplements/mitie)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23980025/)
* [GitHub](https://github.com/ratschlab/MiTie)

#### mtim: an isoform inference tool
Last release: 2013, April, 18th
* [arXiv](https://arxiv.org/abs/1309.5211)
* [GitHub](https://github.com/nicococo/mTIM)

#### NBSplice: a differential isoform usage
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31972288/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/NBSplice.html)

#### NLDMSeq: an isoform quantification tool
Last release: 2016, June, 27th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26475308/)
* [GitHub](https://github.com/PUGEA/NLDMseq)

#### NURD: an isoform quantification tool
Last release: _not found_. Published in 2013.
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/NURD/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23837734/)

#### Oases: a transcriptome assembly tool
Last release: 2011, December, 6th
* [URL](https://www.ebi.ac.uk/~zerbino/oases/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22368243/)

#### PAIRADISE: a paired differential isoform expression
Last release: 2020, June, 9th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32781045/)
* [GitHub](https://github.com/Xinglab/PAIRADISE)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/PAIRADISE.html)

#### PAVfinder: de novo isoforms inference from transcriptome assemblies
Last release: 2020, December, 28th
* [URL](https://www.bcgsc.ca/resources/software/pavfinder)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30200994/)
* [GitHub](https://github.com/bcgsc/pavfinder)

#### PennDiff: a differential isoform usage detection
Last release: 2020, December, 20th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29474557/)
* [GitHub](https://github.com/tigerhu15/PennDiff)

#### PRAM: transcript models inference from pooled RNA-seq
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32958497/)
* [Bioconductor](http://bioconductor.org/packages/release/bioc/html/pram.html)

#### PSGInfer: an isoforms inference and differential analysis tool
Last release: 2013, October, 25th
* [URL](http://deweylab.biostat.wisc.edu/psginfer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23846746/)

#### RNASeqViewer: a visualization tool for gene expression and isoforms
Last release: 2013, October, 14th
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/RNAseqViewer/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24215023/)

#### rnaSPAdes: a transcriptome assembly tool
Last release: 2021, February
* [URL](http://cab.spbu.ru/software/rnaspades/)
* [bioRxiv](https://www.biorxiv.org/content/10.1101/2020.01.24.918482v1)
* [GitHub](https://github.com/ablab/spades)

#### rDiff: a differential isoform expression detection tool
Last release: 2020, June, 19th (the last commit)
* [URL](https://public.bmi.inf.ethz.ch/user/drewe/rdiff/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23585274/)
* [GitHub](https://github.com/ratschlab/rDiff)

#### RSEM: an isoform quantification tool
The variant of the tools is pRSEM: an isoform quantification from a complementary data
Last release: 2020, February, 14th
* [URL RSEM](https://deweylab.github.io/RSEM/)
* [URL pRSEM](https://deweylab.github.io/pRSEM/)
* [PubMed RSEM](https://pubmed.ncbi.nlm.nih.gov/21816040/)
* [PubMed pRSEM](https://pubmed.ncbi.nlm.nih.gov/27405803/)
* [GitHub](https://github.com/deweylab/RSEM)

#### rSeqDiff: a differential isoform expression detection tool
Last release: _not found_. Published in 2013
* [URL](https://jhui2014.github.io/rseqdiff/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24260225/)

#### rSeqNP: a differential isoform expression detection tool
Last release: 
* [URL](https://jhui2014.github.io/rseqnp/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/25717189)

#### Sailfish: an isoform quantification tool
Last release: 2016, April, 14th
* [URL](https://sailfish.readthedocs.io/en/master/sailfish.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24752080/)
* [GitHub](https://github.com/kingsfordgroup/sailfish)

#### salmon: an isoform quantification tool
Last release: 2020, November, 26th
* [URL](https://combine-lab.github.io/salmon/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28263959/)
* [GitHub](https://github.com/COMBINE-lab/salmon)

#### SDEAP: a differential isoform expression detection tool for population data
Last release: 2017, March, 15th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27522083/)
* [GitHub](https://github.com/ewyang089/SDEAP/wiki)

#### Scripture: a transcriptome assembly tool
Last release:
* [URL](http://software.broadinstitute.org/software/scripture/home)
* [PubMed]()

#### SigFuge: a differential isoform expression detection tool
Last release:_ not found_. Published in 2014.
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/SigFuge.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25030904/)

#### SLIDE: an isoform inference and quantification tool
Last release: 2012, May, 17th
* [URL](http://www.stat.ucla.edu/~jingyi.li/software-and-data.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22135461/)

#### SOAPdenovo-trans: a transcriptome assembly tool
Last release: 2017, August, 23rd
* [URL](https://sourceforge.net/projects/soapdenovotrans/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24532719/)
* [GitHub](https://github.com/aquaskyline/SOAPdenovo-Trans)

#### Sparselso: an isoform identification tool
Last release: 2017, October, 11th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28968634/)
* [GitHub](https://github.com/henryxushi/SparseIso)

#### SpliceGrapher: a tool for creating splice graphs and isoforms predicting
Last release: 2014, April, 24th
* [URL](http://splicegrapher.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22293517/)

#### spliceR: a tool for alternative splicing, differential detection and isoform switching
Last release:_ not found_. Published in 2014
* [Bioconductor](http://www.bioconductor.org/packages/2.13/bioc/html/spliceR.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24655717/)

#### StringTie: a transcriptome assembly and isoform quantification tool
Last release: 2021, March, 4th
* [URL](http://ccb.jhu.edu/software/stringtie/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25690850/)

#### SUPPA: an isoform quantification tool
Last release: 2018, February, 22nd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26179515/)
* [PubMed SUPPA2](https://pubmed.ncbi.nlm.nih.gov/29571299/)
* [GitHub](https://github.com/comprna/SUPPA)
* [BitBucker](https://bitbucket.org/regulatorygenomicsupf/suppa/src/master/)

#### tappAS: a functional effect of alternative splicing
Last release: 2021, January, 8th
* [URL](https://app.tappas.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32423416/)

#### TIGAR2: an isoform quantification tool from longer RNA-Seq
Last release: 2015, February, 19th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23821651/)
* [GitHub](https://github.com/nariai/tigar2)

#### TRANS-Abyss: a transcriptome assembly tool
Last release: 2018, February, 19th
* [URL](https://www.bcgsc.ca/resources/software/trans-abyss)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20935650/)
* [GitHub](https://github.com/bcgsc/transabyss)

#### Traph: an isoform quantification tool
Last release: 2014, September, 2nd
* [URL](https://www.cs.helsinki.fi/en/gsa/traph/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23734627/)

#### TRIMD: Iso-Seq, RNA-Seq and deepCAGE data integration for transcriptome resolution (Perl scripts)
Last release: 2020, September, 9th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27407110/)
* [GitHub](https://github.com/flemingtonlab/TRIMD)

#### Trinity: a transcriptome assembly tool
Last release: 2015, January, 24th
* [URL](https://sourceforge.net/projects/trinityrnaseq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21572440/)

#### TSIS: a time-series isoform switches detection tool
Last release: 2018, December, 29th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29028262/)
* [GitHub](https://github.com/wyguo/TSIS)

#### Velvet: a transcriptome assembly tool
Last release: 2018, July, 5th
* [URL](https://www.ebi.ac.uk/~zerbino/velvet/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/18349386/)
* [GitHub](https://github.com/dzerbino/velvet/tree/master)

### Exon-level
#### ChopStitch: an exon identification and splice graph construction out of transcriptome assembly and WGS
Last release: 2018, November, 8th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29300846/)
* [GitHub](https://github.com/bcgsc/ChopStitch)

#### DEXSeq: a differential exon usage tool
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/DEXSeq.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722343/)

#### ExClust: a package for constitutive exons analysis
Part of the sysSeq package.

Last release: 2015, July, 17th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23360225/)
* [GitHub](https://github.com/ellispatrick/sydSeq)

#### ExCluster: a differential exon expression tool
Last release: _not found_. Published in 2020.
* [BioConductor](https://bioconductor.org/packages/release/bioc/html/ExCluster.html)

#### FreePSI: an exon-inclusion ratio quantification tool
Last release: 2017, September, 5th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29136203/)
* [GitHub](https://github.com/JY-Zhou/FreePSI)

#### ggsashimi: an alternative splicing visualization
Last release: 2021, February, 10th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30118475/)
* [GitHub](https://github.com/guigolab/ggsashimi)

#### HMMSplicer: a splice junctions detection
Last release: 2010, November, 25th
* [URL](http://derisilab.ucsf.edu/index.php?software=105)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21079731/)

#### HTSeq: a RNA-Seq quantification tool
The tool can quantify exon expression

Last release: 2020, April, 18th
* [URL](https://htseq.readthedocs.io/en/master/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25260700/)
* [GitHub](https://github.com/htseq/htseq)

#### iGEMS: a model for alternative exon usage detection (R scripts)
Last release: 2016, April, 13th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27095197/)
* [GitHub](https://github.com/iaingallagher/iGEMS_scripts)

#### IntEREst: an intron retention estimation tool
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29642843/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/IntEREst.html)

#### iTAD: a switch-like exons identification (Upon request)
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28542625/)

#### JunctionSeq: a differential exon and junctions usage
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27257077/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/JunctionSeq.html)

#### LeafCutter: an intron quantification and differential analysis
Last release: 2017, September, 26th
* [URL LeafViz](https://leafcutter.shinyapps.io/leafviz/)
* [URL](http://davidaknowles.github.io/leafcutter/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29229983/)
* [GitHub](https://github.com/davidaknowles/leafcutter)

#### MISO: a differential isoform expression tool
The tool also detects differentially regulated exons.
MISO also presents sashimi plots for exon expression visualisation.

Last release: 2019, May, 17th. No longer maintained.
* [URL](http://hollywood.mit.edu/burgelab/miso/)
* [PubMed MISO](https://pubmed.ncbi.nlm.nih.gov/21057496/)
* [PubMed sashimi plots](https://pubmed.ncbi.nlm.nih.gov/25617416/)
* [GitHub](https://github.com/yarden/MISO)

#### Portcullis: a splice junctions detection
Last release: 2018, March, 19th
* [URL Documemntation](https://portcullis.readthedocs.io/en/latest/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6302956/)
* [GitHub](https://github.com/TGAC/portcullis)

#### Read-Splict-Fly: a splice junctions detection (base on Read-Split-Walk and Read-Split-Run)
Last release: 2016, January, 6th
* [URL RSW](http://isu.indstate.edu/ybai2/RSW/index.html)
* [URL RSF](http://bioinf1.indstate.edu:3096/about)
* [PubMed RSW](https://www.ncbi.nlm.nih.gov/pubmed/24991935/)
* [PubMed RSR](https://pubmed.ncbi.nlm.nih.gov/27556805/)
* [GitHub RSR](https://github.com/xuric/read-split-run)
* [GitHub RSF](https://github.com/kinnejeff/read-split-fly)

#### SpliceJumper: a splice junctions detection
Last release: 2015, June, 29th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26678515/)
* [GitHub](https://github.com/simoncchu/SpliceJumper)

#### SpliceLauncher: a splice junctions detection
Last release: 2021, February, 15th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31617569/)
* [GitHub](https://github.com/raphaelleman/SpliceLauncher)

#### SpliceTrap: a tool to quantify exon inclusion levels using paired-end RNA-seq data
Last release: 2013, May, 17th
* [URL](http://rulai.cshl.edu/splicetrap/)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21896509/)

#### SpliceV: a splicing visualization
Last release: 2019, September, 27th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31068132/)
* [GitHub](https://github.com/flemingtonlab/SpliceV)

#### SplicingGraphs: a splicing graphs creation, visualization and quantification tool
Last release: 2020, November, 3rd
* [GitHub](https://github.com/Bioconductor/SplicingGraphs)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/SplicingGraphs.html)

#### TECtool: a terminal codon characterization
Last release: 2018, June, 22th
* [URL](http://tectool.unibas.ch/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30202060/)
* [GitHub](https://github.com/zavolanlab/TECtool)

## sQTL analysis
### GLiMMPS: an sQTL detection
Last release: 2013, March, 12th
* [GutHub](https://github.com/Xinglab/GLiMMPS)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23876401/)

### PVAAS: a tool to identify SNVs associated with aberrant splicing
Last release:
* [URL](http://pvaas.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25573917/)

### sQTLseekeR: an sQTL detection
Last release: 2015, June, 29th
* [GitHub](https://github.com/guigolab/sQTLseekeR)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25140736/)

### SplicePlot: an sQTL visualization
Last release: 2013, October, 15th
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/24363378/)
* [GitHub](https://github.com/wueric/SplicePlot)

### ulfasQTL: an sQTL detection (based on sQTLseekeR)
Last release: *not found*
* [URL](http://bioinfo.au.tsinghua.edu.cn/software/ulfasQTL/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28198669/)

## Splice sites prediction and variations analysis
### CADD-splice: a a varian effect on alternative splicing
Last release: *not found*
* [URL](https://cadd.gs.washington.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33618777/)

### CRYP-SKIP: an effect of exonic mutation
Laste release: 2008
* [URL](https://cryp-skip.img.cas.cz/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/19142208)

### CrypSplice: a cryptic splice variations prediction tool
Last release: 
* [URL](http://liuzlab.org/software/crypsplice/#hoem)

### EBChangePoint: a splice site prediction tool
Last release: 2015, May
* [URL](http://ebchangepoint.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26873932/)

### EX-SKIP: a variant effect prediction on alternative splicing
Last release: 2010
* [URL](https://ex-skip.img.cas.cz/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/21309043)

### GeneSplicer: a splice site prediction
Last release:_ not found_
* [URL](https://www.cbcb.umd.edu/software/GeneSplicer/gene_spl.shtml)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/11222768/)

### HAL: an effect od variants on alternative splicing
Last release: *not found*
* [URL](http://splicing.cs.washington.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26496609/)

### Human Splicing Finder: a splice site prediction (free limited access)
Last release:_ not found_
* [URL](https://www.genomnis.com/access-hsf)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19339519/)

### MaxEntScan: a tool for human 5'splice sites scoring
Last release:_ not found_
* [URL](http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html)

### MIRA: a tool for mutation identification for RNA Alterations
Last release: 2018, July, 3rd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29592900/)
* [GitHub](https://github.com/comprna/MIRA)

### MMSplice: a tool for prediction of effect of genetic variants on alternative splicing
Last release: 2021, February, 20th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30823901/)
* [GitHub](https://github.com/gagneurlab/MMSplice_MTSplice)
* [Kipoi](http://kipoi.org/models/MMSplice/)

### NetAspGene: a splice site prediction in Aspergillus
Last release:_ not found_
* [URL](https://services.healthtech.dtu.dk/service.php?NetAspGene-1.0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/18948220/)

### NetGene2 Server: a splice site prediction
Last release: 2017, May, 30th
* [URL 1](http://www.cbs.dtu.dk/services/NetGene2/)
* [URL 2](https://services.healthtech.dtu.dk/service.php?NetGene2-2.42)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/2067018/)

### NetUTR: a splice site presiction in 5'UTR regions of human genes
Last release: _ not found_
* [URL](https://services.healthtech.dtu.dk/service.php?NetUTR-1.0)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/14960723/)

### NNSPLICE: a splice site prediction by neural network
Last release: 1997, January
* [URL](https://www.fruitfly.org/seq_tools/splice.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/9278062/)

### PEPSI: a prediction of PSI value changes due to genetic variants
Last release: 2019, May, 15th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31074545/)
* [GitHub](https://github.com/rwang916/PEPSI)

### PredPSI-SVR: a prediction of PSI value changes due to genetic variants
Last release: 2020, July, 10th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31070294/)
* [GitHub](https://github.com/chenkenbio/PredPSI-SVR)

### regSNP-intron: a tool which predicts disease-causing probability of human intronic single nucleotide variants (iSNVs)
Last release: 2019, September, 27th
* [URL](https://regsnps-intron.ccbb.iupui.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31779641/)
* [GitHub](https://github.com/yunliu/regsnp_intron)

### rPGA: a hidden splicing variations
Last release: 2016, June, 24th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28766294/)
* [GitHub](https://github.com/Xinglab/rPGA)

### SAVNet: a tool for detection of effect of genetic variants on alternative splicing
Last release: 2019, May, 18th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31687339/)
* [GitHub](https://github.com/friend1ws/SAVNet)

### SiLVA: prediction of synonymous (silent) mutations within the human genome
Last release: 2013, July, 12th
* [URL](http://compbio.cs.toronto.edu/silva/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23736532/)

### SKIPPY: a tool for detection of exonic variants that modulate splicing
Last release: 2011, March, 2nd
* [URL](https://research.nhgri.nih.gov/skippy/download.shtml)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20158892)

### SpliceAI: a tool for detection of effect of genetic variants on alternative splicing
Last release: 2020, March, 7th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30661751/)
* [GitHub](https://github.com/Illumina/SpliceAI)

### Spliceman: a web tool for predition of sequence variations in splicing
Last release:
* [URL](http://fairbrother.biomed.brown.edu/spliceman/index.cgi)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/22328782/)

### SpliceView: a splice site prediction
Last release: *not found*
* [URL](http://bioinfo.itb.cnr.it/~webgene/wwwspliceview.html)

### SPANR: a splicing-based analysis of SNVs
Last release: 
* [URL](http://tools.genes.toronto.edu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/25525159/)

## Alternative splicing regulation
### BPP: a branch point prediction
Last release: 2017, May, 22nd (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28633445/)
* [GitHub](https://github.com/zhqingit/BPP)

### branchpointer: a branch point prediction
Last release: *not found*
* [bioRxiv](https://www.biorxiv.org/content/10.1101/094003v1)
* [Bioconductor](https://www.bioconductor.org/packages/release/bioc/html/branchpointer.html)

### dSreg: a differential splicing and regulation prediction
Last release: 2019 August 7th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31834368/)
* [Bitbucket](https://bitbucket.org/cmartiga/dsreg/src/master/)

### ESEFinder: a web tool for exon splicing enhancers identification
Last release: *not found*
* [URL](http://krainer01.cshl.edu/cgi-bin/tools/ESE3/esefinder.cgi?process=home)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824367/)

### ExonScan: a regulatory exonic motifs identitfication (Upon request)
Last release: *not found*
* [URL](http://hollywood.mit.edu/burgelab/software.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/15607979/)

### ExonSuite: a tool for PUF binding regions analysis
Last release: 2013, January, 13th (the last commit)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/23816174/)
* [GitHub](https://github.com/dilanustek/ExonSuite)

#### LaBranchoR: a branchpoints prediction tool
Last release: 2019, February, 20th
* [URL](http://bejerano.stanford.edu/labranchor/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30224349/)
* [GitHub](https://github.com/jpaggi/labranchor)

### RBPmap: a tool for mapping binding sites of RNA binding proteins
Last release: 2021, February
* [URL](http://rbpmap.technion.ac.il/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24829458/)

### rMAPS2: a tool for RNA-binding protein analysis
Last release: 2020, March, 20th
* [URL](http://rmaps.cecsresearch.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32286627/)

### RNABP: a branchpoint prediction
Last release: 2020
* [URL](http://nsclbio.jbnu.ac.kr/tools/RNABP/)

### SVM-BPfinder: a branchpoints prediction tool
Last release: 2016
* [URL](http://regulatorygenomics.upf.edu/Software/SVM_BP/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21124863/)

## Proteomics
### jcast: takes in alternative splicing events and returns custom protein sequence databases for isoform analysis
Last release: March, 2021 (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31825849/)
* [GitHub](https://github.com/ed-lau/jcast)

### QUILTS: the identification of novel proteins, resulting from single nucleotide variants, splice variants and fusion genes
Last release: *not found*
* [URL](http://openslice.fenyolab.org/cgi-bin/pyquilts_cgi.pl)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26631509/)
* [GitHub](https://github.com/ekawaler/pyQUILTS/) - on a very early stage

### PASS: an alternative splicing detection from proteome
Last release: 2019, April, 29th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31095856/)
* [GitHub](https://github.com/wupengomics/PASS)

### SpliceVista: a tool for splice variant identification and visualization in shotgun proteomics data
Last release: 2017, June 21st (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24692640/)
* [GitHub](https://github.com/yafeng/SpliceVista)

### SPLICIFY: a proteogenomic pipeline for differential splice variant identification
Last release: 2018, May, 28th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28747380/)
* [GitHub](https://github.com/NKI-TGO/SPLICIFY)

## Network analysis
### CoSpliceNet: a co-splicing network tool
Last release: 2017, October, 18th (the last commit)
* [URL](http://delasa.github.io/co-spliceNet/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27793091/)
* [GitHub](https://github.com/Delasa/co-spliceNet)

### DIGGER: a role of alternative splicing in protein-protein interactions
Last release: *not found*
* [URL](https://exbio.wzw.tum.de/digger/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32976589/)

### DomainGraph: effect of alternative splicing on domain-domain interactions
Last release: December, 2018
* [URL](https://domaingraph.bioinf.mpi-inf.mpg.de/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/20513647/)

### n1pas: a alternatevely spliced pathway detection
Last release: 2019, April, 3rd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31143202/)
* [GitHub](https://github.com/grizant/n1pas/tree/master)

### PathwaySplice: a pathway analysis for alternative splicing genes
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29688305/)
* [Bioconductor](https://bioconductor.org/packages/release/bioc/html/PathwaySplice.html)

## Single cell data
### BRIE: an isoform quantification tool
Last release: 2018, June, 3rd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28655331/)
* [GitHub](https://github.com/huangyh09/brie)

### Expedition: an alternative splicing detection tool
Last release: 
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28673540/)
* [GitHub](https://github.com/YeoLab/Expedition)

### SingleSplice: an alternative splicing detection tool
Last release: 2016, July, 7th (the last commit)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26740580/)
* [GitHub](https://github.com/jw156605/SingleSplice)

### VALERIE: visualising splicing at single-cell
Last release: *not found*
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32898151/)
* [CRAN](https://cran.r-project.org/web/packages/VALERIE/index.html)

## Long reads
### deSALT: an alignment tool
Last release: 2020, August, 20th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31842925/)
* [GitHub](https://github.com/ydLiu-HIT/deSALT)

### flair
Last release: 2020, May, 4th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32188845/)
* [GitHub](https://github.com/BrooksLabUCSC/flair)

### scallop-LR: a transcritome assembler
Last release: 2020, May, 22nd
* [bioRxiv](https://www.biorxiv.org/content/10.1101/632703v1)
* [GitHub](https://github.com/Kingsford-Group/scallop/tree/isoseq)

### SpliceHunter
Last release: 2017, March, 23rd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29508290/)
* [Bitbucket](https://bitbucket.org/canzar/splicehunter/src/master/)

## Isoform Function Prediction
### DIFFUSE
Last release: 2020, October, 29th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31510699/)
* [GitHub](https://github.com/haochenucr/DIFFUSE)

## Circular RNAs

### CircAST
Last release: 2020, February, 26th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32007626/)
* [GitHub](https://github.com/xiaofengsong/CircAST)

### CircSplice
Last release: 2019, February, 27th
* [URL](http://gb.whu.edu.cn/CircSplice/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30849979/)
* [GitHub](https://github.com/GeneFeng/CircSplice)

### Ciri
Last release: 2021, March, 15th
* [URL](https://sourceforge.net/projects/ciri/)
* [PubMed 1](https://pubmed.ncbi.nlm.nih.gov/27350239/)
* [PubMed 2](https://pubmed.ncbi.nlm.nih.gov/30660194/)

### FUCHS
Last release: 2020, August, 17th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28265491/)
* [GitHub](https://github.com/dieterich-lab/FUCHS)

## Other
### ACEScan: the algorithm for prediction of conserved alternatively spliced exons from pairs of conserved mouse/human exons (Upon request)
Last release: 2005, July, 10th
* [URL](http://hollywood.mit.edu/burgelab/software.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/15708978/)

### ASimulatoR: RNA-Seq reads with splicing simulation
Last release: 2020, December, 16th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33647976/)
* [GitHub](https://github.com/biomedbigdata/ASimulatoR)

### ASNEO: a detection of alternative splicing neoantigens
Last release: 2020, May, 24th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32697765/)
* [GitHub](https://github.com/bm2-lab/ASNEO)

### BEERS: RNA-Seq reads with splicing simulation
Last release: 2015, December, 9th (the last commit)
* [URL](http://cbil.upenn.edu/BEERS/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21775302/)
* [GitHub](https://github.com/itmat/beers_simulator)

#### Enhanced Integrated Gradients: a method to identify specific features
Last release: 2020, June, 30th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32560708/)
* [Bitbucket](https://bitbucket.org/biociphers/eig/src/master/)

#### ESPRNN: epigenome-based splicing prediction
Last release: 2020, February, 11th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32584815/)
* [GitHub](https://github.com/gersteinlab/esprnn)

### FSOM: a methylation patterns associated with alternative splicing
Last release: 2015, September, 3rd
* [URL](https://sourceforge.net/projects/fsom/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26365234/)

### IsoSel: an isoform phylogenetic analysis tool
Last release: *not found*
* [URL](http://doua.prabi.fr/software/isosel)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28323858/)

### Matt: a down-stream alternative splicing analysis
Last release: 2019, May, 27th
* [URL](http://matt.crg.eu/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30010778/)

### Mirage: multiple sequence isoform alignment
Last release: 2021, January, 21st
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31080963/)
* [GitHub](https://github.com/TravisWheelerLab/Mirage)

### NMD Classifier: a nonsense mediated decay classifier
Last release: 2019, April, 11th
* [URL](https://sourceforge.net/projects/transcriptome-analysis/files/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28369084/)

### OneStopRNAseq: a pipeline for RNA-Seq analysis
Includes rMATS and DEXSeq

Last release: *not found*
* [URL](https://mccb.umassmed.edu/OneStopRNAseq/index.php)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33023248/)

### ORQAS: an alternative splicing quantification tool from RNA-Seq and Ribo-Seq
Last release: 2020, April, 23rd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32286305/)
* [GitHub](https://github.com/comprna/orqas)

### PPLine: a pipeline for SNP calling and gene/isoform quantification
Last release: 2018, March, 26th
* [URL](https://sourceforge.net/projects/ppline/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26147802/)

### PrimerSeq: a tool for RT-PCR primers design that evaluate alternative splicing events
Last release: 2016, November, 18th
* [URL](http://primerseq.sourceforge.net/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24747190/)
* [GitHub](https://github.com/ctokheim/PrimerSeq)

### SEASTAR: an alternative transcription start sites detection tool
Last release: 2017, October, 26th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29546410/)
* [GitHub](https://github.com/Xinglab/SEASTAR)

### SimSpliceEvol: an alternative splicing-aware simulation of biological sequence evolution
Last release: 2019, august, 2nd
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31842741/)
* [GitHub](https://github.com/UdeS-CoBIUS/SimSpliceEvol)

### Shark: aapping-free tool for fishing relevant reads in an RNA-Seq sample
Last release: 2020, July, 6th
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32926128/)
* [GitHub](https://github.com/AlgoLab/shark)

### TraC: a shared between splice variants sequences
Last release: 2018, May, 24th
* [URL](http://labs.pathology.jhu.edu/nauen/trac/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29763731/)
* [GitHub](https://github.com/nauenlab/TraC)

### TranscriptAchilles: a transcript-level biomarkers for cancer
Last release: 2019
* [URL](https://biotecnun.unav.es/app/TranscriptAchilles)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30942869/)
* [GitLab](https://gitlab.com/fcarazo.m/transcriptachilles)

### Transposable element-derived alternative splicing in cancer
Last release: *not found*
* [URL](http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&hubUrl=http://jordan.biology.gatech.edu/teAs/hub.txt)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32075558/)

### UNCOVER: the algorithm for predicting conserved alternatively spliced exons (and retained introns) in orthologous intron pairs (Upon request)
Last release: *not found*
* [URL](http://hollywood.mit.edu/burgelab/software.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16110330/)

## Databases
### AceView: a database of alternative splicing isoforms
Last update: 2012, October, 16th

Organisms: human, mouse, rat, worm, arabidopsis
* [URL](https://www.ncbi.nlm.nih.gov/IEB/Research/Acembly/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16925834/)

### AltTrans: a database of alternative splicing and alternative polyadenilation (archived)
Last update: 2008, September, 5th

Organisms: human, mouse, rat
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/16556303/)
* [FTP](ftp://ftp.ebi.ac.uk/pub/databases/astd)

### APPRIS: a database of principal splice isoforms
Last update: 2021

Organisims: 19 vertebrate and invertebrate genomes
* [URL](http://appris.bioinfo.cnio.es/#/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23161672)

### ASlive: a database for alternative splicing in livestock animals
Last update: 2019, June, 25th

Organisms: cattle, sheep, pig, chicken, horse
* [URL](http://202.195.250.104/lsas/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32000661/)

### ASpedia: a database of human alternative splicing
Last update: 2017, June, 30th

Organisims: human
* [URL](http://combio.snu.ac.kr/aspedia/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29106599/)

### ASPicDB: a database of alternative splicing patterns in human genes
Last update: 2017, May, 4th, 196,446 splicing events	

Organism: human
* [URL](http://srv00.recas.ba.infn.it/ASPicDB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/21051348/)

### CancerSplicingQTL: a database of SNPs that affect alternative splicing
Last update: 2021

Organism: human
* [URL](http://www.cancersplicingqtl-hust.com/#/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30329095/)

### CuAS: a database of alternative splicing in cucumber
Last update: 2021

Organism: cucumber
* [URL](http://cmb.bnu.edu.cn/alt_iso/index.php)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32183712/)

### DBTSS: a database of transcription start sites
Last update: 2017, December, 21st

Organism: human
* [URL](https://dbtss.hgc.jp/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/29126224)

### ExonSkipDB: a functional characterization of skipped exons
Last update: *not found*

Organism: Human
* [URL](https://ccsm.uth.edu/ExonSkipDB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31642488/)

### H-DBAS: a human alternative splicing database
Last update: 2011, December, 26th

Organism: human
* [URL](http://h-invitational.jp/h-dbas/)
* [PubMed](http://www.ncbi.nlm.nih.gov/pubmed/19969536?dopt=Abstract)

### HEXEvent: a database of Human EXon splicing Events
Last update: 2017, September, 29th

Organism: human
* [URL](http://hexevent.mmg.uci.edu/cgi-bin/HEXEvent/HEXEventWEB.cgi)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/23118488/)

### ISOdb: a database of Iso-Seq isoforms
Last update: 2018, March, 5th

Organism: 8 organisms
* [URL](http://isodb.xieslab.org/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/30581839/)

### ISOexpresso: a database of isoform expression in cancer
Last update: 2016, June, 1st

Organism: human
* [URL](http://wiki.tgilab.org/ISOexpresso/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/27519173/)

### Kassiopeia: a database of mutually exclusive exonomes
Last update: _not found_

Organism: human, fruit fly, worm, Arabidopsis
* [URL](https://kassiopeia.motorprotein.de/kassiopeia)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/24507667/)

### LncAS2Cancer: alternative splicing in lncRNAs in cancer
Last update: 2021, February, 8th

Organism: human
* [URL](https://lncrna2as.cd120.com/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/32820322/)

### Pan-cancer repository of validated natural and cryptic mRNA splicing mutations
Last update: *not found*

Organism: human
* [URL](https://validsplicemut.cytognomix.com/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/31275557/)

### Plant Alternative Splicing Database
Last update: 2019

Organism: 15 plant genomes
* [URL](http://proteomics.ysu.edu/altsplice/)

### PlantSPEAD: a splicing-related protein expression database
Last update: 2017

Organisms: plants
* [URL](http://chemyang.ccnu.edu.cn/ccb/database/PlantSPEAD/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/33010087/)

### PolyA_DB: a database of pre-mRNA cleavage and polyA sites
Last update: 2018, August. 107 human samples, 246 mouse samples, 11 rat samples, 9 chicken samples.

Organisms: human, mouse, rat, chicken
* [URL](https://exon.apps.wistar.org/PolyA_DB/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29069441/)

### ProSAS: a database of the effects of alternative splicing events on the structure of the resulting protein isoforms.
Last update: _not found_. Published in 2008.

Organism
* [URL](https://drupal.bio.ifi.lmu.de/forschung/structural-bioinformatics/prosas/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/17933774)

### SpliceAid: a database of experimentally proven RNA target motifs bound by splicing proteins in human
Last update: 2013, February

Organism: human
* [URL](http://www.introni.it/splicing.html)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/19261717/)

### SpliceAid2: a database of human splicing factors expression data and RNA target motifs
Last update: 2013 February:	71 splicing factors, 2,339 RNA binding sites

Organism: human
* [URL](http://193.206.120.249/splicing_tissue.html)
* [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21922594/)

### TassDB: a tandem splice sites database
Last update: _not found_. Published in 2010.

* [URL 1](http://www.tassdb.info/)
* [URL 2](http://tassdb2.leibniz-fli.de/)
* [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/20429909)

### TCGA SpliceSeq: splicing patterns alternation in TCGA
Last update: *not found*
* [URL](https://bioinformatics.mdanderson.org/TCGASpliceSeq/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/26602693/)

### TSVdb: TCGA splicing variants database
Last update: 2020, May, 9th (the last commit)
* [URL](http://www.tsvdb.com/)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/29843604/)
* [GitHub](https://github.com/wenjie1991/TSVdb)

### VastDB: a database of alternative splicing profiles
Last release: 2020, December, 21st
* [URL](https://vastdb.crg.eu/wiki/Main_Page)
* [PubMed](https://pubmed.ncbi.nlm.nih.gov/28855263/)
